<!doctype html><html lang=en-us><head><link rel=preconnect href=https://fonts.googleapis.com><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@700&display=swap" rel=stylesheet><meta charset=utf-8><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><script src=https://ajax.googleapis.com/ajax/libs/jquery/3.6.1/jquery.min.js></script><script>var unis={};$(document).on("change","input[type='checkbox']",function(){if(this.id.startsWith("check_year")){const collection=document.getElementById("main").children;for(child of collection)
{if(child.id.startsWith(this.id.slice(-9)))
{this.checked?child.style.display="initial":child.style.display="none";}}}else if(this.id.startsWith("uni")){var uni=this.id.slice(-7);if(unis[uni]==undefined)unis[uni]=true;this.checked?(unis[uni]=true):(unis[uni]=false);const collection=document.querySelectorAll("[data-uni="+uni+"]");for(child of collection){if(this.checked){child.parentNode.parentNode.setAttribute("style","display:flex !important")}else{disp_off_flag=true;for(child of child.parentNode.querySelectorAll("[data-uni]")){if(unis[child.getAttribute("data-uni")]==undefined)unis[child.getAttribute("data-uni")]=true;if(unis[child.getAttribute("data-uni")]==true){disp_off_flag=false;break;}}
if(disp_off_flag){child.parentNode.parentNode.setAttribute("style","display:none !important");}}}}else if(this.id.startsWith("prj")){var prj=this.id.slice(-4);const collection=document.querySelectorAll("[data-prj="+prj+"]");for(child of collection){(this.checked)?child.setAttribute("style","display:initial !important"):child.setAttribute("style","display:none !important")}}});function clearAll(t){if(t=='year'){$(':input[id^="check_year"]').prop('checked',false);$('div[id^="year_"]').prop('style','display:none;');}else if(t=='uni'){$(':input[id^="uni"]').prop('checked',false);$('p.d-sm-flex').prop('style','display:none!important;');for(let key in unis){unis[key]=false}}else if(t=='prj'){$(':input[id^="prj"]').prop('checked',false);$('div[data-prj]').prop('style','display:none!important;');data-prj}}</script><style>.arrowup{border:solid #fff;border-width:0 3px 3px 0;display:inline-block;padding:5px;transform:rotate(-135deg);-webkit-transform:rotate(-135deg)}#topBtn{display:none;position:fixed;bottom:20px;right:30px;z-index:99;font-size:30px;border:none;outline:none;background-color:#29abe0;color:#fff;cursor:pointer;padding:15px;border-radius:8px;height:10dp}#topBtn:hover{background-color:#1b8dbb}.card-header-custom{cursor:pointer;-webkit-user-select:none;-ms-user-select:none;user-select:none}.clear-widget{color:#747474!important;background-color:#dee2e6;font-size:80%!important}.clear-widget:hover{color:#212529!important;background-color:#c1c9d0;cursor:pointer}</style><!--[if IEMobile]><meta http-equiv=cleartype content="on"><![endif]--><title>Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy - GWF Publications</title><meta name=generator content="Hugo 0.68.3"><link href=/gwf-publications/gwficon.ico rel="shortcut icon" type=image/x-icon><link rel=stylesheet href=/gwf-publications/css/main.min.8976777c0832d068a49d330764e507857027f1efa3b8501cf349b0e2db7410fc.css media=screen><link rel=stylesheet href=https://use.fontawesome.com/releases/v5.7.2/css/all.css integrity=sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr crossorigin=anonymous><link rel=stylesheet href=/gwf-publications/css/academicons.min.css><meta content="Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy" name=citation_title><meta content="Tyson E. Graber" name=citation_author><meta content="Kamya Bhatnagar" name=citation_author><meta content="Elisabeth Mercier" name=citation_author><meta content="Meghan L.M. Fuzzen" name=citation_author><meta content="Patrick M. D'Aoust" name=citation_author><meta content="Huy-Dung Hoang" name=citation_author><meta content="Xin Tian" name=citation_author><meta content="Syeda Tasneem Towhid" name=citation_author><meta content="Luis Gimeno" name=citation_author><meta content="Tommy Alain" name=citation_author><meta content="Ainslie Butler" name=citation_author><meta content="Lawrence Goodridge" name=citation_author><meta content="Mark R. Servos" name=citation_author><meta content="Robert Delatolla" name=citation_author><meta content name=citation_journal_title><meta content="2021" name=citation_publication_date><meta content="https://gwf-uwaterloo.github.io/gwf-publications/G21-47001.pdf" name=citation_pdf_url><meta content="10.1101/2021.02.22.21252041" name=citation_doi><meta property="og:title" content="Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy"><meta property="og:image" content="https://gwf-uwaterloo.github.io/gwf-publications/thumb/G21-62010.jpg"><meta property="og:image:alt" content="First page of paper PDF."><meta property="og:type" content="article"><meta property="og:site_name" content="ACL Anthology"><meta property="og:url" content="https://gwf-uwaterloo.github.io/gwf-publications/G21-47001"><meta property="og:description" content="Tyson E. Graber, Kamya Bhatnagar, Elisabeth Mercier, Meghan L.M. Fuzzen, Patrick M. D'Aoust, Huy-Dung Hoang, Xin Tian, Syeda Tasneem Towhid, Luis Gimeno, Tommy Alain, Ainslie Butler, Lawrence Goodridge, Mark R. Servos, Robert Delatolla. . 2021."><link rel=canonical href=https://gwf-uwaterloo.github.io/gwf-publications/G21-47001></head><body><header style="z-index:1002;top:0;position:sticky;display:block;box-sizing:border-box;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;line-height:1.5;color:#222;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%"><div id=navbar-container style=display:block;overflow:hidden;z-index:10;left:0;margin:0;line-height:2em;white-space:normal;background-color:#000;color:#dfdfdf;height:57px;right:0;top:0><a href=/gwf-publications/ style=outline:none;position:absolute;top:50%;left:50%;max-width:45%;transform:translate(-50%,-50%)><img alt src=/gwf-publications/images/waterloo-logo.svg style="display:flex;flex-shrink:0;transition:opacity 300ms linear 0s;cursor:pointer;width:139.273px;max-width:100%;vertical-align:middle"></a></div></header><div id=main-container class=container><section id=main><div><h2 id=title><a href=https://gwf-uwaterloo.github.io/gwf-publications/G21-47001>Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy</a></h2><p class=lead><a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=uni0140>Tyson E. Graber</a>,
<a href=/gwf-publications/people/K/Kamya-Bhatnagar/ data-uni=uni0026>Kamya Bhatnagar</a>,
<a href=/gwf-publications/people/E/Elisabeth-Mercier/ data-uni=uni0026>Elisabeth Mercier</a>,
<a href=/gwf-publications/people/M/Meghan-LM-Fuzzen/ data-uni=uni0002>Meghan L.M. Fuzzen</a>,
<a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=uni0026>Patrick M. D'Aoust</a>,
<a href=/gwf-publications/people/H/Huy-Dung-Hoang/ data-uni=uni0140>Huy-Dung Hoang</a>,
<a href=/gwf-publications/people/X/Xin-Tian/ data-uni=uni0886>Xin Tian</a>,
<a href=/gwf-publications/people/S/Syeda-Tasneem-Towhid/ data-uni=uni0026>Syeda Tasneem Towhid</a>,
<a href=/gwf-publications/people/L/Luis-Gimeno/ data-uni=uni0885>Luis Gimeno</a>,
<a href=/gwf-publications/people/T/Tommy-Alain/ data-uni=uni0140>Tommy Alain</a>,
<a href=/gwf-publications/people/A/Ainslie-Butler/ data-uni=uni0884>Ainslie Butler</a>,
<a href=/gwf-publications/people/L/Lawrence-Goodridge/ data-uni=uni0005>Lawrence Goodridge</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0026>Robert Delatolla</a></p></div><hr><div class="row acl-paper-details"><div class="col col-lg-10 order-2"><div class="card bg-light mb-2 mb-lg-3"><div class="card-body acl-abstract"><h5 class=card-title>Abstract</h5><span>Abstract The coronavirus disease 2019 (COVID-19) pandemic caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has claimed millions of lives to date. Antigenic drift has resulted in viral variants with putatively greater transmissibility, virulence, or both. Early and near real-time detection of these variants of concern (VOC) and the ability to accurately follow their incidence and prevalence in communities is wanting. Wastewater-based epidemiology (WBE), which uses nucleic acid amplification tests to detect viral fragments, is a faithful proxy of COVID-19 incidence and prevalence, and thus offers the potential to monitor VOC viral load in a given population. Here, we describe and validate a primer extension PCR strategy targeting a signature mutation in the N gene of SARS-CoV-2. This allows quantification of the proportional expression of B.1.1.7 versus non-B.1.1.7 alleles in wastewater without the need to employ quantitative RT-PCR standard curves. We show that the wastewater B.1.1.7 profile correlates with its clinical counterpart and benefits from a near real-time and facile data collection and reporting pipeline. This assay can be quickly implemented within a current SARS-CoV-2 WBE framework with minimal cost; allowing early and contemporaneous estimates of B.1.1.7 community transmission prior to, or in lieu of, clinical screening and identification. Our study demonstrates that this strategy can provide public health units with an additional and much needed tool to rapidly triangulate VOC incidence/prevalence with high sensitivity and lineage specificity.</span></div></div><dl><dt>Cite:</dt><dd><span id=citeACL>Tyson E. Graber, Kamya Bhatnagar, Elisabeth Mercier, Meghan L.M. Fuzzen, Patrick M. D'Aoust, Huy-Dung Hoang, Xin Tian, Syeda Tasneem Towhid, Luis Gimeno, Tommy Alain, Ainslie Butler, Lawrence Goodridge, Mark R. Servos, and Robert Delatolla. 2021. Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy.</span><button type=button class="btn btn-clipboard btn-secondary btn-sm d-none ml-2" data-clipboard-target=#citeACL><i class="far fa-clipboard"></i></button></dd><dt class=acl-button-row>Copy Citation:</dt><dd class=acl-button-row><button type=button class="btn btn-clipboard-outside btn-secondary btn-sm d-none" data-clipboard-target=#citeBibtexContent><i class="far fa-clipboard pr-2"></i>BibTeX</button>
<button type=button class="btn btn-clipboard-outside btn-secondary btn-sm d-none" data-clipboard-target=#citeMarkdownContent><i class="far fa-clipboard pr-2"></i>Markdown</button>
<button type=button class="btn btn-clipboard-outside btn-secondary btn-sm d-none" data-clipboard-target=#citeModsContent><i class="far fa-clipboard pr-2"></i>MODS XML</button>
<button type=button class="btn btn-clipboard-outside btn-secondary btn-sm d-none" data-clipboard-target=#citeEndnoteContent><i class="far fa-clipboard pr-2"></i>Endnote</button>
<button type=button class="btn btn-secondary btn-sm" data-toggle=modal data-target=#citeModal>More options…</button></dd></dl></div><div class=acl-paper-link-block><a class="btn btn-secondary" title="Open dialog for exporting citations" data-toggle=modal data-target=#citeModal href=#><i class="fas fa-quote-left"></i><span class=pl-2>Cite</span></a>
<a class="btn btn-secondary" href="https://www.semanticscholar.org/search?q=Near+real-time+determination+of+B.1.1.7+in+proportion+to+total+SARS-CoV-2+viral+load+in+wastewater+using+an+allele-specific+primer+extension+PCR+strategy" title="Search for 'Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy' on Semantic Scholar"><i class="ai ai-semantic-scholar"></i><span class="pl-sm-2 d-none d-sm-inline">Search</span></a></div></div><hr><div class="modal fade" id=citeModal tabindex=-1 role=dialog aria-labelledby=citeModalLabel aria-hidden=true><div class="modal-dialog modal-lg" role=document><div class=modal-content><div class=modal-header><h5 class=modal-title id=citeModalLabel>Export citation</h5><button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><ul class="nav nav-tabs mb-2" id=citeFormats role=tablist><li class=nav-item><a class="nav-link active" data-toggle=list href=#citeBibtex role=tab aria-controls=citeBibtex aria-selected=true>BibTeX</a></li><li class=nav-item><a class=nav-link data-toggle=list href=#citeMods role=tab aria-controls=citeMods aria-selected=false>MODS XML</a></li><li class=nav-item><a class=nav-link data-toggle=list href=#citeEndnote role=tab aria-controls=citeEndnote aria-selected=false>Endnote</a></li><li class=nav-item><a class=nav-link data-toggle=list href=#citeMarkdown role=tab aria-controls=citeMarkdown aria-selected=false>Preformatted</a></li></ul><div class=tab-content id=citeFormatsContent><div class="tab-pane active" id=citeBibtex role=tabpanel><pre id=citeBibtexContent class="bg-light border p-2" style=max-height:50vh>@article{Graber-2021-Near,
    title = &#34;Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy&#34;,
    author = &#34;Graber, Tyson E.  and
      Bhatnagar, Kamya  and
      Mercier, Elisabeth  and
      Fuzzen, Meghan L.M.  and
      D&#39;Aoust, Patrick M.  and
      Hoang, Huy-Dung  and
      Tian, Xin  and
      Towhid, Syeda Tasneem  and
      Gimeno, Luis  and
      Alain, Tommy  and
      Butler, Ainslie  and
      Goodridge, Lawrence  and
      Servos, Mark R.  and
      Delatolla, Robert&#34;,
    journal = &#34;&#34;,
    year = &#34;2021&#34;,
    publisher = &#34;Copernicus GmbH&#34;,
    url = &#34;https://gwf-uwaterloo.github.io/gwf-publications/G21-47001&#34;,
    doi = &#34;10.1101/2021.02.22.21252041&#34;,
    abstract = &#34;Abstract The coronavirus disease 2019 (COVID-19) pandemic caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has claimed millions of lives to date. Antigenic drift has resulted in viral variants with putatively greater transmissibility, virulence, or both. Early and near real-time detection of these variants of concern (VOC) and the ability to accurately follow their incidence and prevalence in communities is wanting. Wastewater-based epidemiology (WBE), which uses nucleic acid amplification tests to detect viral fragments, is a faithful proxy of COVID-19 incidence and prevalence, and thus offers the potential to monitor VOC viral load in a given population. Here, we describe and validate a primer extension PCR strategy targeting a signature mutation in the N gene of SARS-CoV-2. This allows quantification of the proportional expression of B.1.1.7 versus non-B.1.1.7 alleles in wastewater without the need to employ quantitative RT-PCR standard curves. We show that the wastewater B.1.1.7 profile correlates with its clinical counterpart and benefits from a near real-time and facile data collection and reporting pipeline. This assay can be quickly implemented within a current SARS-CoV-2 WBE framework with minimal cost; allowing early and contemporaneous estimates of B.1.1.7 community transmission prior to, or in lieu of, clinical screening and identification. Our study demonstrates that this strategy can provide public health units with an additional and much needed tool to rapidly triangulate VOC incidence/prevalence with high sensitivity and lineage specificity.&#34;,
}
</pre><div class="modal-footer pb-1"><a class="btn btn-secondary" href=/gwf-publications/G21-62010.bib><i class="fas fa-download pr-2"></i>Download as File</a>
<button class="btn btn-clipboard btn-primary d-none" data-clipboard-target=#citeBibtexContent><i class="far fa-clipboard pr-2"></i>Copy to Clipboard</button></div></div><div class=tab-pane id=citeMods role=tabpanel><pre id=citeModsContent class="bg-light border p-2" style=max-height:50vh>﻿&lt;?xml version=&#34;1.0&#34; encoding=&#34;UTF-8&#34;?&gt;
&lt;modsCollection xmlns=&#34;http://www.loc.gov/mods/v3&#34;&gt;
&lt;mods ID=&#34;Graber-2021-Near&#34;&gt;
    &lt;titleInfo&gt;
        &lt;title&gt;Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy&lt;/title&gt;
    &lt;/titleInfo&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Tyson&lt;/namePart&gt;
        &lt;namePart type=&#34;given&#34;&gt;E&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Graber&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Kamya&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Bhatnagar&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Elisabeth&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Mercier&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Meghan&lt;/namePart&gt;
        &lt;namePart type=&#34;given&#34;&gt;L.M.&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Fuzzen&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Patrick&lt;/namePart&gt;
        &lt;namePart type=&#34;given&#34;&gt;M&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;D’Aoust&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Huy-Dung&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Hoang&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Xin&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Tian&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Syeda&lt;/namePart&gt;
        &lt;namePart type=&#34;given&#34;&gt;Tasneem&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Towhid&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Luis&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Gimeno&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Tommy&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Alain&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Ainslie&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Butler&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Lawrence&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Goodridge&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Mark&lt;/namePart&gt;
        &lt;namePart type=&#34;given&#34;&gt;R&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Servos&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;Robert&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Delatolla&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;originInfo&gt;
        &lt;dateIssued&gt;2021&lt;/dateIssued&gt;
    &lt;/originInfo&gt;
    &lt;typeOfResource&gt;text&lt;/typeOfResource&gt;
    &lt;genre authority=&#34;bibutilsgt&#34;&gt;journal article&lt;/genre&gt;
    &lt;relatedItem type=&#34;host&#34;&gt;
        &lt;titleInfo&gt;
            &lt;title/&gt;
        &lt;/titleInfo&gt;
        &lt;originInfo&gt;
            &lt;issuance&gt;continuing&lt;/issuance&gt;
            &lt;publisher&gt;Copernicus GmbH&lt;/publisher&gt;
        &lt;/originInfo&gt;
        &lt;genre authority=&#34;marcgt&#34;&gt;periodical&lt;/genre&gt;
        &lt;genre authority=&#34;bibutilsgt&#34;&gt;academic journal&lt;/genre&gt;
    &lt;/relatedItem&gt;
    &lt;abstract&gt;Abstract The coronavirus disease 2019 (COVID-19) pandemic caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has claimed millions of lives to date. Antigenic drift has resulted in viral variants with putatively greater transmissibility, virulence, or both. Early and near real-time detection of these variants of concern (VOC) and the ability to accurately follow their incidence and prevalence in communities is wanting. Wastewater-based epidemiology (WBE), which uses nucleic acid amplification tests to detect viral fragments, is a faithful proxy of COVID-19 incidence and prevalence, and thus offers the potential to monitor VOC viral load in a given population. Here, we describe and validate a primer extension PCR strategy targeting a signature mutation in the N gene of SARS-CoV-2. This allows quantification of the proportional expression of B.1.1.7 versus non-B.1.1.7 alleles in wastewater without the need to employ quantitative RT-PCR standard curves. We show that the wastewater B.1.1.7 profile correlates with its clinical counterpart and benefits from a near real-time and facile data collection and reporting pipeline. This assay can be quickly implemented within a current SARS-CoV-2 WBE framework with minimal cost; allowing early and contemporaneous estimates of B.1.1.7 community transmission prior to, or in lieu of, clinical screening and identification. Our study demonstrates that this strategy can provide public health units with an additional and much needed tool to rapidly triangulate VOC incidence/prevalence with high sensitivity and lineage specificity.&lt;/abstract&gt;
    &lt;identifier type=&#34;citekey&#34;&gt;Graber-2021-Near&lt;/identifier&gt;
    &lt;identifier type=&#34;doi&#34;&gt;10.1101/2021.02.22.21252041&lt;/identifier&gt;
    &lt;location&gt;
        &lt;url&gt;https://gwf-uwaterloo.github.io/gwf-publications/G21-47001&lt;/url&gt;
    &lt;/location&gt;
    &lt;part&gt;
        &lt;date&gt;2021&lt;/date&gt;
    &lt;/part&gt;
&lt;/mods&gt;
&lt;/modsCollection&gt;
</pre><div class="modal-footer pb-1"><a class="btn btn-secondary" href=/gwf-publications/G21-62010.xml><i class="fas fa-download pr-2"></i>Download as File</a>
<button class="btn btn-clipboard btn-primary d-none" data-clipboard-target=#citeModsContent><i class="far fa-clipboard pr-2"></i>Copy to Clipboard</button></div></div><div class=tab-pane id=citeEndnote role=tabpanel><pre id=citeEndnoteContent class="bg-light border p-2" style=max-height:50vh>﻿%0 Journal Article
%T Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy
%A Graber, Tyson E.
%A Bhatnagar, Kamya
%A Mercier, Elisabeth
%A Fuzzen, Meghan L.M.
%A D’Aoust, Patrick M.
%A Hoang, Huy-Dung
%A Tian, Xin
%A Towhid, Syeda Tasneem
%A Gimeno, Luis
%A Alain, Tommy
%A Butler, Ainslie
%A Goodridge, Lawrence
%A Servos, Mark R.
%A Delatolla, Robert
%D 2021
%I Copernicus GmbH
%F Graber-2021-Near
%X Abstract The coronavirus disease 2019 (COVID-19) pandemic caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has claimed millions of lives to date. Antigenic drift has resulted in viral variants with putatively greater transmissibility, virulence, or both. Early and near real-time detection of these variants of concern (VOC) and the ability to accurately follow their incidence and prevalence in communities is wanting. Wastewater-based epidemiology (WBE), which uses nucleic acid amplification tests to detect viral fragments, is a faithful proxy of COVID-19 incidence and prevalence, and thus offers the potential to monitor VOC viral load in a given population. Here, we describe and validate a primer extension PCR strategy targeting a signature mutation in the N gene of SARS-CoV-2. This allows quantification of the proportional expression of B.1.1.7 versus non-B.1.1.7 alleles in wastewater without the need to employ quantitative RT-PCR standard curves. We show that the wastewater B.1.1.7 profile correlates with its clinical counterpart and benefits from a near real-time and facile data collection and reporting pipeline. This assay can be quickly implemented within a current SARS-CoV-2 WBE framework with minimal cost; allowing early and contemporaneous estimates of B.1.1.7 community transmission prior to, or in lieu of, clinical screening and identification. Our study demonstrates that this strategy can provide public health units with an additional and much needed tool to rapidly triangulate VOC incidence/prevalence with high sensitivity and lineage specificity.
%R 10.1101/2021.02.22.21252041
%U https://gwf-uwaterloo.github.io/gwf-publications/G21-47001
%U https://doi.org/10.1101/2021.02.22.21252041

</pre><div class="modal-footer pb-1"><a class="btn btn-secondary" href=/gwf-publications/G21-62010.endf><i class="fas fa-download pr-2"></i>Download as File</a>
<button class="btn btn-clipboard btn-primary d-none" data-clipboard-target=#citeEndnoteContent><i class="far fa-clipboard pr-2"></i>Copy to Clipboard</button></div></div><div class=tab-pane id=citeMarkdown role=tabpanel><h5>Markdown (Informal)</h5><p id=citeMarkdownContent class="text-monospace small bg-light border p-2">[Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy](https://gwf-uwaterloo.github.io/gwf-publications/G21-47001) (Graber et al., GWF 2021)</p><ul class=mt-2><li><a href=https://gwf-uwaterloo.github.io/gwf-publications/G21-47001>Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy</a> (Graber et al., GWF 2021)</li></ul><h5>ACL</h5><ul class=mt-2><li id=citeACLstyleContent>Tyson E. Graber, Kamya Bhatnagar, Elisabeth Mercier, Meghan L.M. Fuzzen, Patrick M. D'Aoust, Huy-Dung Hoang, Xin Tian, Syeda Tasneem Towhid, Luis Gimeno, Tommy Alain, Ainslie Butler, Lawrence Goodridge, Mark R. Servos, and Robert Delatolla. 2021. Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy.</li></ul><div class="modal-footer pb-1"><button type=button class="btn btn-clipboard btn-primary d-none" data-clipboard-target=#citeMarkdownContent><i class="far fa-clipboard pr-2"></i>Copy Markdown to Clipboard</button>
<button type=button class="btn btn-clipboard btn-primary d-none" data-clipboard-target=#citeACLstyleContent><i class="far fa-clipboard pr-2"></i>Copy ACL to Clipboard</button></div></div></div></div></div></div></div></section></div><footer class="bg-gradient-light py-2 py-xl-3 mt-3 mt-md-4 mt-xl-5"><div class=container><p class="text-muted small px-1">Global Water Futures Publications!</p></div></footer><script src=https://code.jquery.com/jquery-3.3.1.slim.min.js integrity=sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js integrity=sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut crossorigin=anonymous></script><script src=https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js integrity=sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k crossorigin=anonymous></script><script>$(function(){$('[data-toggle="tooltip"]').tooltip();if($("#toggle-all-abstracts")){$("#toggle-all-abstracts").click(function(){var target=$("#toggle-all-abstracts");target.attr("disabled",true);if(target.attr("data-toggle-state")=="hide"){$(".abstract-collapse").collapse('show');target.attr("data-toggle-state","show");}else{$(".abstract-collapse").collapse('hide');target.attr("data-toggle-state","hide");}
target.attr("disabled",false);});$("#toggle-all-abstracts").attr("disabled",false);}})</script><script src=/gwf-publications/js/clipboard.min.js></script><script>$(document).ready(function(){if(ClipboardJS.isSupported()){success_fn=function(e){var src=$(e.trigger);src.toggleClass("btn-success");src.children("i").toggleClass("far fa-clipboard fas fa-clipboard-check");e.clearSelection();setTimeout(function(){src.toggleClass("btn-success");src.children("i").toggleClass("far fa-clipboard fas fa-clipboard-check");},2000);};var clipboard=new ClipboardJS(".btn-clipboard");clipboard.on('success',success_fn);$(".btn-clipboard").removeClass("d-none");var clipboard_outside=new ClipboardJS(".btn-clipboard-outside",{text:function(trigger){var target=trigger.getAttribute("data-clipboard-target");return $(target).text();}});clipboard_outside.on('success',success_fn);$(".btn-clipboard-outside").removeClass("d-none");}});</script></body></html>