<!doctype html><html lang=en-us><head><link rel=preconnect href=https://fonts.googleapis.com><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@700&display=swap" rel=stylesheet><meta charset=utf-8><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><script src=https://ajax.googleapis.com/ajax/libs/jquery/3.6.1/jquery.min.js></script><script>var unis={};$(document).on("change","input[type='checkbox']",function(){if(this.id.startsWith("check_year")){const collection=document.getElementById("main").children;for(child of collection)
{if(child.id.startsWith(this.id.slice(-9)))
{this.checked?child.style.display="initial":child.style.display="none";}}}else if(this.id.startsWith("uni")){var uni=this.id.slice(-7);if(unis[uni]==undefined)unis[uni]=true;this.checked?(unis[uni]=true):(unis[uni]=false);const collection=document.querySelectorAll("[data-uni="+uni+"]");for(child of collection){if(this.checked){child.parentNode.parentNode.setAttribute("style","display:flex !important")}else{disp_off_flag=true;for(child of child.parentNode.querySelectorAll("[data-uni]")){if(unis[child.getAttribute("data-uni")]==undefined)unis[child.getAttribute("data-uni")]=true;if(unis[child.getAttribute("data-uni")]==true){disp_off_flag=false;break;}}
if(disp_off_flag){child.parentNode.parentNode.setAttribute("style","display:none !important");}}}}else if(this.id.startsWith("prj")){var prj=this.id.slice(-4);const collection=document.querySelectorAll("[data-prj="+prj+"]");for(child of collection){(this.checked)?child.setAttribute("style","display:initial !important"):child.setAttribute("style","display:none !important")}}});function clearAll(t){if(t=='year'){$(':input[id^="check_year"]').prop('checked',false);$('div[id^="year_"]').prop('style','display:none;');}else if(t=='uni'){$(':input[id^="uni"]').prop('checked',false);$('p.d-sm-flex').prop('style','display:none!important;');for(let key in unis){unis[key]=false}}else if(t=='prj'){$(':input[id^="prj"]').prop('checked',false);$('div[data-prj]').prop('style','display:none!important;');data-prj}}</script><style>.arrowup{border:solid #fff;border-width:0 3px 3px 0;display:inline-block;padding:5px;transform:rotate(-135deg);-webkit-transform:rotate(-135deg)}#topBtn{display:none;position:fixed;bottom:20px;right:30px;z-index:99;font-size:30px;border:none;outline:none;background-color:#29abe0;color:#fff;cursor:pointer;padding:15px;border-radius:8px;height:10dp}#topBtn:hover{background-color:#1b8dbb}.card-header-custom{cursor:pointer;-webkit-user-select:none;-ms-user-select:none;user-select:none}.clear-widget{color:#747474!important;background-color:#dee2e6;font-size:80%!important}.clear-widget:hover{color:#212529!important;background-color:#c1c9d0;cursor:pointer}</style><!--[if IEMobile]><meta http-equiv=cleartype content="on"><![endif]--><title>Jun Xu - GWF Publications</title><meta name=generator content="Hugo 0.68.3"><link href=/gwf-publications-dev-wiir/gwficon.ico rel="shortcut icon" type=image/x-icon><link rel=stylesheet href=/gwf-publications-dev-wiir/css/main.min.8976777c0832d068a49d330764e507857027f1efa3b8501cf349b0e2db7410fc.css media=screen><link rel=stylesheet href=https://use.fontawesome.com/releases/v5.7.2/css/all.css integrity=sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr crossorigin=anonymous><link rel=stylesheet href=/gwf-publications-dev-wiir/css/academicons.min.css></head><body><header style="z-index:1002;top:0;position:sticky;display:block;box-sizing:border-box;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;line-height:1.5;color:#222;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%"><div id=navbar-container style=display:block;overflow:hidden;z-index:10;left:0;margin:0;line-height:2em;white-space:normal;background-color:#000;color:#dfdfdf;height:57px;right:0;top:0><a href=/gwf-publications-dev-wiir/ style=outline:none;position:absolute;top:50%;left:50%;max-width:45%;transform:translate(-50%,-50%)><img alt src=/gwf-publications-dev-wiir/images/waterloo-logo.svg style="display:flex;flex-shrink:0;transition:opacity 300ms linear 0s;cursor:pointer;width:139.273px;max-width:100%;vertical-align:middle"></a></div></header><div id=main-container class=container><section id=main><h2 id=title><span class=font-weight-normal>Jun</span> <span class=font-weight-bold>Xu</span></h2><hr><div class=row><div class=col-lg-9><h4>2020</h4><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications-dev-wiir/G20-95001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G20-95001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G20-95001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications-dev-wiir/G20-95001/>Integrated assessment of west coast of South Korea by use of benthic bacterial community structure as determined by eDNA, concentrations of contaminants, and in vitro bioassays</a></strong><br><a href=/gwf-publications-dev-wiir/people/A/Aslan-Hwanhwi-Lee/ data-uni=uni0001>Aslan Hwanhwi Lee</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jung-Hyun-Lee/ data-uni=uni0173>Jung-Hyun Lee</a>,
<a href=/gwf-publications-dev-wiir/people/S/Seongjin-Hong/ data-uni=uni0300>Seongjin Hong</a>,
<a href=/gwf-publications-dev-wiir/people/B/Bong-Oh-Kwon/ data-uni=uni1332>Bong-Oh Kwon</a>,
<a href=/gwf-publications-dev-wiir/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications-dev-wiir/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jong-Seong-Khim/ data-uni=uni1021>Jong Seong Khim</a><br><a href=/gwf-publications-dev-wiir/volumes/G20-95/ class=text-muted>Environment International, Volume 137</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G20-95001><div class="card-body p-3 small">During the past few decades, contamination of sediments by persistent toxic substances (PTSs) has been observed in estuarine and coastal areas on the west coast of South Korea. The contaminants are suspected to cause toxicities in aquatic biota, but little is known about their ecological effects, particularly on benthic microbial communities. In this study, an eDNA-based assessment was applied along with classic assessments of exposure, such as chemistry and in vitro bioassays, to evaluate condition of benthic bacterial communities subjected to PTSs. Two strategies were adopted for the study. One was to conduct a comprehensive assessment in space (by comparing seawater and freshwater sites at five coastal regions) and in time (by following change over a 5-y period). Although we found that bacterial composition varied among and within years, some phyla, such as Proteobacteria (28.7%), Actinobacteria (13.1%), Firmicutes (12.7%), and Chloroflexi (12.5%) were consistently dominated across the study regions. Certain bacterial groups, such as Firmicutes and Verrucomicrobia have been linked to contamination at some sites in the study area and at specific points in time. Bacterial communities were not significantly correlated with salinity or AhR- and ER-mediated potencies, whereas concentrations of PAHs, APs, and certain metals (Cd and Hg) exhibited significant associations to the structure of bacterial communities at the phylum level. In fact, the relative abundance of microbes in the phylum Planctomycetes was significantly and negatively correlated with concentrations of PAHs and metals. Thus, the relative abundance of Planctomycetes could be used as an indicator of sedimentary contamination by PAHs and/or metals. Based on our correlation analyses, Cd and ER-mediated potencies were associated more with bacterial abundances at the class taxonomic level than were other PTSs and metals. Overall, the eDNA-based assessment was useful by augmenting more traditional measures of exposure and responses in a sediment triad approach and has potential as a more rapid screening tool.</div></div><h4>2019</h4><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications-dev-wiir/G19-92001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G19-92001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G19-92001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications-dev-wiir/G19-92001/>Sedimentary DNA reveals over 150â€¯years of ecosystem change by human activities in Lake Chao, China</a></strong><br><a href=/gwf-publications-dev-wiir/people/F/Fei-Long-Li/ data-uni=uni1212>Fei-Long Li</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications-dev-wiir/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications-dev-wiir/people/J/Ji-Zhong-Wang/ data-uni=uni1482>Ji-Zhong Wang</a><br><a href=/gwf-publications-dev-wiir/volumes/G19-92/ class=text-muted>Environment International, Volume 133</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G19-92001><div class="card-body p-3 small">Understanding the extent and directionality of the impact of human activities on ecosystems is directly related to their management and protection. However, the lack of historical data limits our understanding of ecosystem changes with long-term exposure to human activities. Recently, lake sedimentary DNA (sedDNA) has become a powerful tool for revealing changes in ecosystems at the century and millennium scales. Here, we used sedDNA to reveal the dynamic of the microbial community (including bacteria and micro-eukaryotes) in Lake Chao over the past 150 years, and further explored the effects of long-term nutrient and heavy metal loads on these communities. Our data show that nutrient and heavy metal loads in Lake Chao have increased by ca. 2 to 4-fold since the 1960s. In response, the community structure, diversity, and ecological network of bacteria and micro-eukaryotes changed significantly during the 1960s, the 1980s and the 2010s. Importantly, community structure was more sensitive to human activities than diversity. We also found that the relative abundance of some taxa associated with nitrification and algal blooms (e.g., taxa in Nitrospira sp., Peridinales) has increased ca. 100-fold since the 1960s. Nutrient could better explain the variation in the bacterial community (ca. twice as much as heavy metal), while heavy metal explained micro-eukaryotes better (ca. 3 or 5-fold as much as nutrient). In particular, based on parsimonious models from distance-based linear model (distLM), we further identified that Pb is the key factor affecting the bacterial and micro-eukaryotes community in Lake Chao in addition to nutrient. Our study reveals the impacts of long-term human activities on lake ecosystems from multiple perspectives of nutrient and heavy metal loads, community structure, diversity and ecological network, these findings will contribute to the management and conservation of lakes in the future.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications-dev-wiir/G19-188001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G19-188001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G19-188001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications-dev-wiir/G19-188001/>Acid mine drainage affects the diversity and metal resistance gene profile of sediment bacterial community along a river</a></strong><br><a href=/gwf-publications-dev-wiir/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications-dev-wiir/people/S/Song-Tang/ data-uni=uni0385>Song Tang</a>,
<a href=/gwf-publications-dev-wiir/people/M/Mao-Wang/ data-uni=uni1292>Mao Wang</a>,
<a href=/gwf-publications-dev-wiir/people/W/Weimin-Sun/ data-uni=uni1293>Weimin Sun</a>,
<a href=/gwf-publications-dev-wiir/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications-dev-wiir/people/H/Hui-Peng/ data-uni=uni0420>Hui Peng</a>,
<a href=/gwf-publications-dev-wiir/people/A/Ai-Min-Zhong/ data-uni=uni0231>Ai-Min Zhong</a>,
<a href=/gwf-publications-dev-wiir/people/H/Hongling-Liu/ data-uni=uni0153>Hongling Liu</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications-dev-wiir/people/H/Hongxia-Yu/ data-uni=uni0153>Hongxia Yu</a>,
<a href=/gwf-publications-dev-wiir/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a>,
<a href=/gwf-publications-dev-wiir/people/M/Markus-Hecker/ data-uni=uni0001>Markus Hecker</a><br><a href=/gwf-publications-dev-wiir/volumes/G19-188/ class=text-muted>Chemosphere, Volume 217</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G19-188001><div class="card-body p-3 small">Acid mine drainage (AMD) is one of the most hazardous byproducts of some types of mining. However, research on how AMD affects the bacterial community structure of downstream riverine ecosystems and the distribution of metal resistance genes (MRGs) along pollution gradient is limited. Comprehensive geochemical and high-throughput next-generation sequencing analyses can be integrated to characterize spatial distributions and MRG profiles of sediment bacteria communities along the AMD-contaminated Hengshi River. We found that (1) diversities of bacterial communities significantly and gradually increased along the river with decreasing contamination, suggesting community composition reflected changes in geochemical conditions; (2) relative abundances of phyla Proteobacteria and genus Halomonas and Planococcaceae that function in metal reduction decreased along the AMD gradient; (3) low levels of sediment salinity, sulfate, aquatic lead (Pb), and cadmium (Cd) were negatively correlated with bacterial diversity despite pH was in a positive manner with diversity; and (4) arsenic (As) and copper (Cu) resistance genes corresponded to sediment concentrations of As and Cu, respectively. Altogether, our findings offer initial insight into the distribution patterns of sediment bacterial community structure, diversity and MRGs along a lotic ecosystem contaminated by AMD, and the factors that affect them.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications-dev-wiir/G19-188001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G19-188001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G19-188001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications-dev-wiir/G19-188001/>Acid mine drainage affects the diversity and metal resistance gene profile of sediment bacterial community along a river</a></strong><br><a href=/gwf-publications-dev-wiir/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications-dev-wiir/people/S/Song-Tang/ data-uni=uni0385>Song Tang</a>,
<a href=/gwf-publications-dev-wiir/people/M/Mao-Wang/ data-uni=uni1292>Mao Wang</a>,
<a href=/gwf-publications-dev-wiir/people/W/Weimin-Sun/ data-uni=uni1293>Weimin Sun</a>,
<a href=/gwf-publications-dev-wiir/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications-dev-wiir/people/H/Hui-Peng/ data-uni=uni0420>Hui Peng</a>,
<a href=/gwf-publications-dev-wiir/people/A/Ai-Min-Zhong/ data-uni=uni0231>Ai-Min Zhong</a>,
<a href=/gwf-publications-dev-wiir/people/H/Hongling-Liu/ data-uni=uni0153>Hongling Liu</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications-dev-wiir/people/H/Hongxia-Yu/ data-uni=uni0153>Hongxia Yu</a>,
<a href=/gwf-publications-dev-wiir/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a>,
<a href=/gwf-publications-dev-wiir/people/M/Markus-Hecker/ data-uni=uni0001>Markus Hecker</a><br><a href=/gwf-publications-dev-wiir/volumes/G19-188/ class=text-muted>Chemosphere, Volume 217</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G19-188001><div class="card-body p-3 small">Acid mine drainage (AMD) is one of the most hazardous byproducts of some types of mining. However, research on how AMD affects the bacterial community structure of downstream riverine ecosystems and the distribution of metal resistance genes (MRGs) along pollution gradient is limited. Comprehensive geochemical and high-throughput next-generation sequencing analyses can be integrated to characterize spatial distributions and MRG profiles of sediment bacteria communities along the AMD-contaminated Hengshi River. We found that (1) diversities of bacterial communities significantly and gradually increased along the river with decreasing contamination, suggesting community composition reflected changes in geochemical conditions; (2) relative abundances of phyla Proteobacteria and genus Halomonas and Planococcaceae that function in metal reduction decreased along the AMD gradient; (3) low levels of sediment salinity, sulfate, aquatic lead (Pb), and cadmium (Cd) were negatively correlated with bacterial diversity despite pH was in a positive manner with diversity; and (4) arsenic (As) and copper (Cu) resistance genes corresponded to sediment concentrations of As and Cu, respectively. Altogether, our findings offer initial insight into the distribution patterns of sediment bacterial community structure, diversity and MRGs along a lotic ecosystem contaminated by AMD, and the factors that affect them.</div></div><h4>2018</h4><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications-dev-wiir/G18-67001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G18-67001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G18-67001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications-dev-wiir/G18-67001/>Application of Environmental DNA Metabarcoding for Predicting Anthropogenic Pollution in Rivers</a></strong><br><a href=/gwf-publications-dev-wiir/people/F/Fei-Long-Li/ data-uni=uni1212>Fei-Long Li</a>,
<a href=/gwf-publications-dev-wiir/people/Y/Ying-Peng/ data-uni=uni1213>Ying Peng</a>,
<a href=/gwf-publications-dev-wiir/people/W/Wendi-Fang/ data-uni=uni0153>Wendi Fang</a>,
<a href=/gwf-publications-dev-wiir/people/F/Florian-Altermatt/ data-uni=uni0099>Florian Altermatt</a>,
<a href=/gwf-publications-dev-wiir/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jianghua-Yang/ data-uni=uni0153>Jianghua Yang</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a><br><a href=/gwf-publications-dev-wiir/volumes/G18-67/ class=text-muted>Environmental Science & Technology</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G18-67001><div class="card-body p-3 small">Rivers are among the most threatened freshwater ecosystems, and anthropogenic activities are affecting both river structures and water quality. While assessing the organisms can provide a comprehensive measure of a river's ecological status, it is limited by the traditional morphotaxonomy-based biomonitoring. Recent advances in environmental DNA (eDNA) metabarcoding allow to identify prokaryotes and eukaryotes in one sequencing run, and could thus allow unprecedented resolution. Whether such eDNA-based data can be used directly to predict the pollution status of rivers as a complementation of environmental data remains unknown. Here we used eDNA metabarcoding to explore the main stressors of rivers along which community structure changes, and to identify the method's potential for predicting pollution status based on eDNA data. We showed that a broad range of taxa in bacterial, protistan, and metazoan communities could be profiled with eDNA. Nutrients were the main driving stressor affecting communities' structure, alpha diversity, and the ecological network. We specifically observed that the relative abundance of indicative OTUs was significantly correlated with nutrient levels. These OTUs data could be used to predict the nutrient status up to 79% accuracy on testing data sets. Thus, our study gives a novel approach to predicting the pollution status of rivers by eDNA data.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications-dev-wiir/G18-131001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G18-131001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G18-131001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications-dev-wiir/G18-131001/>In situ microbiota distinguished primary anthropogenic stressor in freshwater sediments</a></strong><br><a href=/gwf-publications-dev-wiir/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications-dev-wiir/people/T/Tilman-Floehr/ data-uni=uni0113>Tilman Floehr</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications-dev-wiir/people/H/Hongxia-Xiao/ data-uni=uni0113>Hongxia Xiao</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jianghua-Yang/ data-uni=uni0153>Jianghua Yang</a>,
<a href=/gwf-publications-dev-wiir/people/P/Pu-Xia/ data-uni=uni1018>Pu Xia</a>,
<a href=/gwf-publications-dev-wiir/people/G/G-Allen-Burton/ data-uni=uni0017>G. Allen Burton</a>,
<a href=/gwf-publications-dev-wiir/people/H/Henner-Hollert/ data-uni=uni0309>Henner Hollert</a><br><a href=/gwf-publications-dev-wiir/volumes/G18-131/ class=text-muted>Environmental Pollution, Volume 239</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G18-131001><div class="card-body p-3 small">Conventional assessment and evaluation of sediment quality are based on laboratory-based ecotoxicological and chemical measurements with lack of concern for ecological relevance. Microbiotas in sediment are responsive to pollutants and can be used as alternative ecological indicators of sediment pollutants; however, the linkage between the microbial ecology and ecotoxicological endpoints in response to sediment contamination has been poorly evaluated. Here, in situ microbiotas from the Three Gorges Reservoir (TGR) area of the Yangtze River were characterized by DNA metabarcoding approaches, and then, changes of in situ microbiotas were compared with the ecotoxicological endpoint, aryl hydrocarbon receptor (AhR) mediated activity, and level of polycyclic aromatic hydrocarbons (PAHs) in sediments. PAHs and organic pollutant mixtures mediating AhR activity had different effects on the structures of microbiotas. Specifically, Shannon indices of protistan communities were negatively correlated with the levels of AhR mediated activity and PAHs. The sediment AhR activity was positively correlated with the relative abundance of prokaryotic Acetobacteraceae, but had a negative correlation with protistan Oxytrichidae. Furthermore, a quantitative classification model was built to predict the level of AhR activity based on the relative abundances of Acetobacteraceae and Oxytrichidae. These results suggested that in situ Protista communities could provide a useful tool for monitoring and assessing ecological stressors. The observed responses of microbial community provided supplementary evidence to support that the AhR-active pollutants, such as PAHs, were the primary stressors of the aquatic community in TGR area.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications-dev-wiir/G18-132001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G18-132001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G18-132001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications-dev-wiir/G18-132001/>eDNA-based bioassessment of coastal sediments impacted by an oil spill</a></strong><br><a href=/gwf-publications-dev-wiir/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jianghua-Yang/ data-uni=uni0153>Jianghua Yang</a>,
<a href=/gwf-publications-dev-wiir/people/S/Seon-Jin-Kim/ data-uni=uni0591>Seon-Jin Kim</a>,
<a href=/gwf-publications-dev-wiir/people/S/Seongjin-Hong/ data-uni=uni0300>Seongjin Hong</a>,
<a href=/gwf-publications-dev-wiir/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a>,
<a href=/gwf-publications-dev-wiir/people/U/Un-Hyuk-Yim/ data-uni=uni1019>Un Hyuk Yim</a>,
<a href=/gwf-publications-dev-wiir/people/W/Won-Heum-Shim/ data-uni=uni1020>Won Heum Shim</a>,
<a href=/gwf-publications-dev-wiir/people/H/Hongxia-Yu/ data-uni=uni0153>Hongxia Yu</a>,
<a href=/gwf-publications-dev-wiir/people/J/Jong-Seong-Khim/ data-uni=uni1021>Jong Seong Khim</a><br><a href=/gwf-publications-dev-wiir/volumes/G18-132/ class=text-muted>Environmental Pollution, Volume 238</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G18-132001><div class="card-body p-3 small">Oil spills offshore can cause long-term ecological effects on coastal marine ecosystems. Despite their important ecological roles in the cycling of energy and nutrients in food webs, effects on bacteria, protists or arthropods are often neglected. Environmental DNA (eDNA) metabarcoding was applied to characterize changes in the structure of micro- and macro-biota communities of surface sediments over a 7-year period since the occurrence of Hebei Spirit oil spill on December 7, 2007. Alterations in diversities and structures of micro- and macro-biota were observed in the contaminated area where concentrations of polycyclic aromatic hydrocarbons were greater. Successions of bacterial, protists and metazoan communities revealed long-term ecological effects of residual oil. Residual oil dominated the largest cluster of the community-environment association network. Presence of bacterial families (Aerococcaceae and Carnobacteriaceae) and the protozoan family (Platyophryidae) might have conferred sensitivity of communities to oil pollution. Hydrocarbon-degrading bacterial families (Anaerolinaceae, Desulfobacteraceae, Helicobacteraceae and Piscirickettsiaceae) and algal family (Araphid pennate) were resistant to adverse effects of spilt oil. The protistan family (Subulatomonas) and arthropod families (Folsomia, Sarcophagidae Opomyzoidea, and Anomura) appeared to be positively associated with residual oil pollution. eDNA metabarcoding can provide a powerful tool for assessing effects of anthropogenic pollution, such as oil spills on sediment communities and its long-term trends in coastal marine environments.</div></div></div><div class=col-lg-3><a class="btn btn-lg btn-secondary btn-block mb-2" href="https://www.semanticscholar.org/search?q=Jun+Xu" title="Search for 'Jun Xu' on Semantic Scholar"><i class="ai ai-semantic-scholar"></i><span class=pl-sm-2>Search</span></a><div class=row><div class="col-12 col-md-6 col-lg-12"><div class=card><h5 class=card-header>Co-authors</h5><ul class="list-group list-group-flush list-group-compact"><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/Y/Yuwei-Xie/ class=align-middle data-uni=uni0001>Yuwei Xie</a>
<span class="badge badge-secondary align-middle ml-2">7</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/J/John-P-Giesy/ class=align-middle data-uni=uni0106>John P. Giesy</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/J/Jianghua-Yang/ class=align-middle data-uni=uni0153>Jianghua Yang</a>
<span class="badge badge-secondary align-middle ml-2">3</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/H/Hongxia-Yu/ class=align-middle data-uni=uni0153>Hongxia Yu</a>
<span class="badge badge-secondary align-middle ml-2">3</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/F/Fei-Long-Li/ class=align-middle data-uni=uni1212>Fei-Long Li</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class="list-group-item list-group-toggle-btn py-1" data-toggle=collapse data-target=#more-coauthors aria-expanded=false aria-controls=more-coauthors>show all...</li><div class="collapse border-top" id=more-coauthors><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/S/Seongjin-Hong/ class=align-middle data-uni=uni0300>Seongjin Hong</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/J/Jong-Seong-Khim/ class=align-middle data-uni=uni1021>Jong Seong Khim</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/S/Song-Tang/ class=align-middle data-uni=uni0385>Song Tang</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/M/Mao-Wang/ class=align-middle data-uni=uni1292>Mao Wang</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/W/Weimin-Sun/ class=align-middle data-uni=uni1293>Weimin Sun</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/H/Hui-Peng/ class=align-middle data-uni=uni0420>Hui Peng</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/A/Ai-Min-Zhong/ class=align-middle data-uni=uni0231>Ai-Min Zhong</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/H/Hongling-Liu/ class=align-middle data-uni=uni0153>Hongling Liu</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/M/Markus-Hecker/ class=align-middle data-uni=uni0001>Markus Hecker</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/Y/Ying-Peng/ class=align-middle data-uni=uni1213>Ying Peng</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/W/Wendi-Fang/ class=align-middle data-uni=uni0153>Wendi Fang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/F/Florian-Altermatt/ class=align-middle data-uni=uni0099>Florian Altermatt</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/T/Tilman-Floehr/ class=align-middle data-uni=uni0113>Tilman Floehr</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/H/Hongxia-Xiao/ class=align-middle data-uni=uni0113>Hongxia Xiao</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/P/Pu-Xia/ class=align-middle data-uni=uni1018>Pu Xia</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/G/G-Allen-Burton/ class=align-middle data-uni=uni0017>G. Allen Burton</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/H/Henner-Hollert/ class=align-middle data-uni=uni0309>Henner Hollert</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/S/Seon-Jin-Kim/ class=align-middle data-uni=uni0591>Seon-Jin Kim</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/U/Un-Hyuk-Yim/ class=align-middle data-uni=uni1019>Un Hyuk Yim</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/W/Won-Heum-Shim/ class=align-middle data-uni=uni1020>Won Heum Shim</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/J/Ji-Zhong-Wang/ class=align-middle data-uni=uni1482>Ji-Zhong Wang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/A/Aslan-Hwanhwi-Lee/ class=align-middle data-uni=uni0001>Aslan Hwanhwi Lee</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/J/Jung-Hyun-Lee/ class=align-middle data-uni=uni0173>Jung-Hyun Lee</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications-dev-wiir/people/B/Bong-Oh-Kwon/ class=align-middle data-uni=uni1332>Bong-Oh Kwon</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li></div></ul></div></div><div class="col-12 col-md-6 col-lg-12"><div class="card my-2 my-md-0 my-lg-2"><h5 class=card-header>Venues</h5><ul class="list-group list-group-flush list-group-compact"><li class=list-group-item><a href=/gwf-publications-dev-wiir/venues/gwf/ class=align-middle>GWF</a><span class="badge badge-secondary align-middle ml-2">7</span></li></ul></div></div></div></div></div></section></div><footer class="bg-gradient-light py-2 py-xl-3 mt-3 mt-md-4 mt-xl-5"><div class=container><p class="text-muted small px-1">Global Water Futures Publications!</p></div></footer><script src=https://code.jquery.com/jquery-3.3.1.slim.min.js integrity=sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js integrity=sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut crossorigin=anonymous></script><script src=https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js integrity=sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k crossorigin=anonymous></script><script>$(function(){$('[data-toggle="tooltip"]').tooltip();if($("#toggle-all-abstracts")){$("#toggle-all-abstracts").click(function(){var target=$("#toggle-all-abstracts");target.attr("disabled",true);if(target.attr("data-toggle-state")=="hide"){$(".abstract-collapse").collapse('show');target.attr("data-toggle-state","show");}else{$(".abstract-collapse").collapse('hide');target.attr("data-toggle-state","hide");}
target.attr("disabled",false);});$("#toggle-all-abstracts").attr("disabled",false);}})</script></body></html>