<!doctype html><html lang=en-us><head><link rel=preconnect href=https://fonts.googleapis.com><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@700&display=swap" rel=stylesheet><meta charset=utf-8><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><script src=https://ajax.googleapis.com/ajax/libs/jquery/3.6.1/jquery.min.js></script><script>var unis={};$(document).on("change","input[type='checkbox']",function(){if(this.id.startsWith("check_year")){const collection=document.getElementById("main").children;for(child of collection)
{if(child.id.startsWith(this.id.slice(-9)))
{this.checked?child.style.display="initial":child.style.display="none";}}}else if(this.id.startsWith("uni")){var uni=this.id.slice(-7);if(unis[uni]==undefined)unis[uni]=true;this.checked?(unis[uni]=true):(unis[uni]=false);const collection=document.querySelectorAll("[data-uni="+uni+"]");for(child of collection){if(this.checked){child.parentNode.parentNode.setAttribute("style","display:flex !important")}else{disp_off_flag=true;for(child of child.parentNode.querySelectorAll("[data-uni]")){if(unis[child.getAttribute("data-uni")]==undefined)unis[child.getAttribute("data-uni")]=true;if(unis[child.getAttribute("data-uni")]==true){disp_off_flag=false;break;}}
if(disp_off_flag){child.parentNode.parentNode.setAttribute("style","display:none !important");}}}}else if(this.id.startsWith("prj")){var prj=this.id.slice(-4);const collection=document.querySelectorAll("[data-prj="+prj+"]");for(child of collection){(this.checked)?child.setAttribute("style","display:initial !important"):child.setAttribute("style","display:none !important")}}});function clearAll(t){if(t=='year'){$(':input[id^="check_year"]').prop('checked',false);$('div[id^="year_"]').prop('style','display:none;');}else if(t=='uni'){$(':input[id^="uni"]').prop('checked',false);$('p.d-sm-flex').prop('style','display:none!important;');for(let key in unis){unis[key]=false}}else if(t=='prj'){$(':input[id^="prj"]').prop('checked',false);$('div[data-prj]').prop('style','display:none!important;');data-prj}}</script><style>.arrowup{border:solid #fff;border-width:0 3px 3px 0;display:inline-block;padding:5px;transform:rotate(-135deg);-webkit-transform:rotate(-135deg)}#topBtn{display:none;position:fixed;bottom:20px;right:30px;z-index:99;font-size:30px;border:none;outline:none;background-color:#29abe0;color:#fff;cursor:pointer;padding:15px;border-radius:8px;height:10dp}#topBtn:hover{background-color:#1b8dbb}.card-header-custom{cursor:pointer;-webkit-user-select:none;-ms-user-select:none;user-select:none}.clear-widget{color:#747474!important;background-color:#dee2e6;font-size:80%!important}.clear-widget:hover{color:#212529!important;background-color:#c1c9d0;cursor:pointer}</style><!--[if IEMobile]><meta http-equiv=cleartype content="on"><![endif]--><title>Yuwei Xie - GWF Publications</title><meta name=generator content="Hugo 0.68.3"><link href=/gwf-publications/gwficon.ico rel="shortcut icon" type=image/x-icon><link rel=stylesheet href=/gwf-publications/css/main.min.8976777c0832d068a49d330764e507857027f1efa3b8501cf349b0e2db7410fc.css media=screen><link rel=stylesheet href=https://use.fontawesome.com/releases/v5.7.2/css/all.css integrity=sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr crossorigin=anonymous><link rel=stylesheet href=/gwf-publications/css/academicons.min.css></head><body><header style="z-index:1002;top:0;position:sticky;display:block;box-sizing:border-box;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;line-height:1.5;color:#222;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%"><div id=navbar-container style=display:block;overflow:hidden;z-index:10;left:0;margin:0;line-height:2em;white-space:normal;background-color:#000;color:#dfdfdf;height:57px;right:0;top:0><a href=/gwf-publications/ style=outline:none;position:absolute;top:50%;left:50%;max-width:45%;transform:translate(-50%,-50%)><img alt src=/gwf-publications/images/waterloo-logo.svg style="display:flex;flex-shrink:0;transition:opacity 300ms linear 0s;cursor:pointer;width:139.273px;max-width:100%;vertical-align:middle"></a></div></header><div id=main-container class=container><section id=main><h2 id=title><span class=font-weight-normal>Yuwei</span> <span class=font-weight-bold>Xie</span></h2><hr><div class=row><div class=col-lg-9><h4>2022</h4><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-17001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-17001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-17001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-17001/>RNA metabarcoding helps reveal zooplankton community response to environmental stressors</a></strong><br><a href=/gwf-publications/people/P/Phillip-J-Ankley/ data-uni=uni0001>Phillip J. Ankley</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/S/Sonya-M-Havens/ data-uni=uni0082>Sonya M. Havens</a>,
<a href=/gwf-publications/people/L/Lisa-E-Peters/ data-uni=uni0013>Lisa E. Peters</a>,
<a href=/gwf-publications/people/L/Lauren-Timlick/ data-uni=uni0013>Lauren Timlick</a>,
<a href=/gwf-publications/people/J/Jos%C3%A9-Luis-Rodr%C3%ADguez-Gil/ data-uni=uni0283>José Luis Rodríguez-Gil</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a>,
<a href=/gwf-publications/people/V/Vince-P-Palace/ data-uni=uni0013>Vince P. Palace</a><br><a href=/gwf-publications/volumes/G22-17/ class=text-muted>Environmental Pollution, Volume 292</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-17001><div class="card-body p-3 small">DNA metabarcoding can provide a high-throughput and rapid method for characterizing responses of communities to environmental stressors. However, within bulk samples, DNA metabarcoding hardly distinguishes live from the dead organisms. Here, both DNA and RNA metabarcoding were applied and compared in experimental freshwater mesocosms conducted for assessment of ecotoxicological responses of zooplankton communities to remediation treatment until 38 days post oil-spill. Furthermore, a novel indicator of normalized vitality (NV), sequence counts of RNA metabarcoding normalized by that of DNA metabarcoding, was developed for assessment of ecological responses. DNA and RNA metabarcoding detected similar taxa richness and rank of relative abundances. Both DNA and RNA metabarcoding demonstrated slight shifts in measured α-diversities in response to treatments. NV presented relatively greater magnitudes of differential responses of community compositions to treatments compared to DNA or RNA metabarcoding. NV declined from the start of the experiment (3 days pre-spill) to the end (38 days post-spill). NV also differed between Rotifer and Arthropoda, possibly due to differential life histories and sizes of organisms. NV could be a useful indicator for characterizing ecological responses to anthropogenic influence; however, the biology of target organisms and subsequent RNA production need to be considered. • RNA normalized by DNA metabarcoding functions as normalized vitality. • Normalized vitality reflected temporal dynamics of zooplankton communities. • Normalized vitality revealed greater community differences between treatments. • Rotifer had greatest normalized vitality compared to Arthropoda.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-19001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-19001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-19001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-19001/>Absorption and elimination of per and poly-fluoroalkyl substances substitutes in salmonid species after pre-fertilization exposure</a></strong><br><a href=/gwf-publications/people/S/Shu-Su/ data-uni=uni0231>Shu Su</a>,
<a href=/gwf-publications/people/P/Paul-W-Jones/ data-uni=uni0408>Paul W. Jones</a>,
<a href=/gwf-publications/people/J/Jason-C-Raine/ data-uni=uni0001>Jason C. Raine</a>,
<a href=/gwf-publications/people/Z/Zilin-Yang/ data-uni=uni0929>Zilin Yang</a>,
<a href=/gwf-publications/people/Y/Yufeng-Gong/ data-uni=uni0008>Yufeng Gong</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/J/Jie-Tang/ data-uni=uni0053>Jie Tang</a>,
<a href=/gwf-publications/people/C/Chao-Wang/ data-uni=uni0037>Chao Wang</a>,
<a href=/gwf-publications/people/X/Xiaoli-Zhao/ data-uni=uni0864>Xiaoli Zhao</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a><br><a href=/gwf-publications/volumes/G22-19/ class=text-muted>Science of The Total Environment, Volume 814</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-19001><div class="card-body p-3 small">Due to their relatively large production and few restrictions on uses, novel substitutes for historically used per and poly-fluoroalkyl substances (PFAS) are being used and accumulating in the environment. However, due to a lack of information on their toxicological properties their hazards and risks are hard to estimate. Before fertilization, oocytes of two salmonid species, Arctic Char (Salvelinus alpinus) and Rainbow Trout (Oncorhynchus mykiss), were exposed to three PFAS substances used as substitutes for traditional PFAS, PFBA, PFBS or GenX or two archetypical, historically used, longer-chain PFAS, PFOA and PFOS. Exposed oocytes were subsequently fertilized, incubated and were sampled during several developmental stages, until swim-up. All five PFAS were accumulated into egg yolks with similar absorption rates, and their concentrations in egg yolks were less than respective concentrations in/on egg chorions. Rapid elimination of the five PFAS was observed during the first 3 days after fertilization. Thereafter, amounts of PFOS and PFOA were stable until swim-up, while PFBA, PFBS and GenX were further eliminated during development from one month after the fertilization to swim-up. In these two salmonid species, PFBA, PFBS and GenX were eliminated faster than were PFOS or PFOA.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-20001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-20001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-20001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-20001/>Effects of aqueous fluoxetine exposure on gut microbiome of adult Pimephales promelas</a></strong><br><a href=/gwf-publications/people/A/Alana-Suevelin-Weber/ data-uni=uni0865>Alana Suevelin Weber</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/J/Jonathan-K-Challis/ data-uni=uni0019>Jonathan K. Challis</a>,
<a href=/gwf-publications/people/A/Abigail-DeBofsky/ data-uni=uni0001>Abigail DeBofsky</a>,
<a href=/gwf-publications/people/P/Phillip-J-Ankley/ data-uni=uni0001>Phillip J. Ankley</a>,
<a href=/gwf-publications/people/M/Markus-Hecker/ data-uni=uni0001>Markus Hecker</a>,
<a href=/gwf-publications/people/P/Paul-W-Jones/ data-uni=uni0408>Paul W. Jones</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a><br><a href=/gwf-publications/volumes/G22-20/ class=text-muted>Science of The Total Environment, Volume 813</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-20001><div class="card-body p-3 small">The microbiome of the gut is vital for homeostasis of hosts with its ability to detoxify and activate toxicants, as well as signal to the immune and nervous systems. However, in the field of environmental toxicology, the gut microbiome has only recently been identified as a measurable indicator for exposure to environmental pollutants. Antidepressants found in effluents of wastewater treatment plants and surface waters have been shown to exhibit antibacterial-like properties in vitro, where some bacteria are known to express homologous proteins that bind antidepressants in vertebrates. Therefore, it has been hypothesized that exposure to antidepressant drugs might affect gut microbiota of aquatic organisms. In this study, the common antidepressant, fluoxetine, was investigated to determine whether it can modulate the gut microbiome of adult fathead minnows. A 28-day, sub-chronic, static renewal exposure was performed with nominal fluoxetine concentrations of 0.01, 10 or 100 μg/L. Using 16S rRNA amplicon sequencing, shifts among the gut-associated microbiota were observed in individuals exposed to the greatest concentration, with greater effects observed in females. These changes were associated with a decrease in relative proportions of commensal bacteria, which can be important for health of fish including bacteria essential for fatty acid oxidation, and an increase in relative proportions of pathogenic bacteria associated with inflammation. Results demonstrate, for the first time, how antidepressants found in some aquatic environments can influence gut microbiota of fishes.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-21001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-21001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-21001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-21001/>RNA in Municipal Wastewater Reveals Magnitudes of COVID-19 Outbreaks across Four Waves Driven by SARS-CoV-2 Variants of Concern</a></strong><br><a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/J/Jonathan-K-Challis/ data-uni=uni0019>Jonathan K. Challis</a>,
<a href=/gwf-publications/people/F/Femi-Francis-Oloye/ data-uni=uni0866>Femi Francis Oloye</a>,
<a href=/gwf-publications/people/M/Mohsen-Asadi/ data-uni=uni0439>Mohsen Asadi</a>,
<a href=/gwf-publications/people/J/Jenna-Cantin/ data-uni=uni0019>Jenna Cantin</a>,
<a href=/gwf-publications/people/M/Markus-Brinkmann/ data-uni=uni0001>Markus Brinkmann</a>,
<a href=/gwf-publications/people/K/Kerry-N-McPhedran/ data-uni=uni0006>Kerry N. McPhedran</a>,
<a href=/gwf-publications/people/N/Natacha-S-Hogan/ data-uni=uni0001>Natacha S. Hogan</a>,
<a href=/gwf-publications/people/M/Mike-Sadowski/ data-uni=uni0867>Mike Sadowski</a>,
<a href=/gwf-publications/people/P/Paul-W-Jones/ data-uni=uni0408>Paul W. Jones</a>,
<a href=/gwf-publications/people/C/Chrystal-Landgraff/ data-uni=uni0005>Chrystal Landgraff</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=uni0466>Chand S. Mangat</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a><br><a href=/gwf-publications/volumes/G22-21/ class=text-muted>ACS ES&T Water, Volume 2, Issue 11</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-21001><div class="card-body p-3 small">There are no standardized protocols for quantifying severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in wastewater to date, especially for population normalization. Here, a pipeline was developed, applied, and assessed to quantify SARS-CoV-2 and key variants of concern (VOCs) RNA in wastewater at Saskatoon, Canada. Normalization approaches using recovery ratio and extraction efficiency, wastewater parameters, or population indicators were assessed by comparing to daily numbers of new cases. Viral load was positively correlated with daily new cases reported in the sewershed. Wastewater surveillance (WS) had a lead time of approximately 7 days, which indicated surges in the number of new cases. WS revealed the variant α and δ driving the third and fourth wave, respectively. The adjustment with the recovery ratio and extraction efficiency improved the correlation between viral load and daily new cases. Normalization of viral concentration to concentrations of the artificial sweetener acesulfame K improved the trend of viral load during the Christmas and New Year holidays when populations were dynamic and variable. Acesulfame K performed better than pepper mild mottle virus, creatinine, and ammonia for population normalization. Hence, quality controls to characterize recovery ratios and extraction efficiencies and population normalization with acesulfame are promising for precise WS programs supporting decision-making in public health.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-110001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-110001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-110001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-110001/>16S rRNA metabarcoding unearths responses of rare gut microbiome of fathead minnows exposed to benzo[a]pyrene</a></strong><br><a href=/gwf-publications/people/A/Abigail-DeBofsky/ data-uni=uni0001>Abigail DeBofsky</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/J/Jonathan-K-Challis/ data-uni=uni0019>Jonathan K. Challis</a>,
<a href=/gwf-publications/people/P/Phillip-J-Ankley/ data-uni=uni0001>Phillip J. Ankley</a>,
<a href=/gwf-publications/people/M/Markus-Brinkmann/ data-uni=uni0001>Markus Brinkmann</a>,
<a href=/gwf-publications/people/P/Paul-W-Jones/ data-uni=uni0408>Paul W. Jones</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a><br><a href=/gwf-publications/volumes/G22-110/ class=text-muted>Science of The Total Environment, Volume 807</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-110001><div class="card-body p-3 small">Activities of gut microbiomes are often overlooked in assessments of ecotoxicological effects of environmental contaminants. Effects of the polycyclic aromatic hydrocarbon, benzo[a]pyrene (BaP) on active gut microbiomes of juvenile fathead minnows (Pimephales promelas) were investigated. Fish were exposed for two weeks, to concentrations of 0, 1, 10, 100, or 1000 μg BaP g-1 in the diet. The active gut microbiome was characterized using 16S rRNA metabarcoding to determine its response to dietary exposure of BaP. BaP reduced alpha-diversity at the greatest exposure concentrations. Additionally, exposure to BaP altered community composition of active microbiome and resulted in differential proportion of taxa associated with hydrocarbon degradation and fish health. Neighborhood selection networks of active microbiomes were not reduced with greater concentrations of BaP, which suggests ecological resistance and/or resilience of gut microbiota. The active gut microbiome had a similar overall biodiversity as that of the genomic gut microbiota, but had a distinct composition from that of the 16S rDNA profile. Responses of alpha- and beta-diversities of the active microbiome to BaP exposure were consistent with that of genomic microbiomes. Normalized activity of microbiome via the ratio of rRNA to rDNA abundance revealed rare taxa that became active or dormant due to exposure to BaP. These differences highlight the need to assess both 16S rDNA and rRNA metabarcoding to fully derive bacterial compositional changes resulting from exposure to contaminants.</div></div><h4>2021</h4><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-39001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-39001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-39001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-39001/>Using zooplankton metabarcoding to assess the efficacy of different techniques to clean-up an oil-spill in a boreal lake</a></strong><br><a href=/gwf-publications/people/P/Phillip-J-Ankley/ data-uni=uni0001>Phillip J. Ankley</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/T/Tyler-Black/ data-uni=uni0029>Tyler Black</a>,
<a href=/gwf-publications/people/A/Abigail-DeBofsky/ data-uni=uni0001>Abigail DeBofsky</a>,
<a href=/gwf-publications/people/M/McKenzie-Perry/ data-uni=uni0013>McKenzie Perry</a>,
<a href=/gwf-publications/people/M/Mike-Paterson/ data-uni=uni0913>Mike Paterson</a>,
<a href=/gwf-publications/people/M/Mark-A-Hanson/ data-uni=uni0914>Mark A. Hanson</a>,
<a href=/gwf-publications/people/S/Scott-N-Higgins/ data-uni=uni0283>Scott N. Higgins</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a>,
<a href=/gwf-publications/people/V/Vince-P-Palace/ data-uni=uni0013>Vince P. Palace</a><br><a href=/gwf-publications/volumes/G21-39/ class=text-muted>Aquatic Toxicology, Volume 236</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-39001><div class="card-body p-3 small">Abstract Regulators require adequate information to select best practices with less ecosystem impacts for remediation of freshwater ecosystems after oil spills. Zooplankton are valuable indicators of aquatic ecosystem health as they play pivotal roles in biochemical cycles while stabilizing food webs. Compared with morphological identification, metabarcoding holds promise for cost-effective, high-throughput, and benchmarkable biomonitoring of zooplankton communities. The objective of this study was to apply DNA and RNA metabarcoding of zooplankton for ecotoxicological assessment and compare it with traditional morphological identification in experimental shoreline enclosures in a boreal lake. These identification methods were also applied in context of assessing response of the zooplankton community exposed to simulated spills of diluted bitumen (dilbit), with experimental remediation practices (enhanced monitored natural recovery and shoreline cleaner application). Metabarcoding detected boreal zooplankton taxa up to the genus level, with a total of 24 shared genera, and while metabarcoding-based relative abundance served as an acceptable proxy for biomass inferred by morphological identification (ρ ≥ 0.52). Morphological identification determined zooplankton community composition changes due to treatments at 11 days post-spill (PERMANOVA, p = 0.0143) while metabarcoding methods indicated changes in zooplankton richness and communities at 38 days post-spill (T-test, p</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-40001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-40001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-40001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-40001/>Life Cycle Exposure to Environmentally Relevant Concentrations of Diphenyl Phosphate (DPhP) Inhibits Growth and Energy Metabolism of Zebrafish in a Sex-Specific Manner</a></strong><br><a href=/gwf-publications/people/Q/Qi-Liang-Chen/ data-uni=uni0915>Qi-Liang Chen</a>,
<a href=/gwf-publications/people/X/Xiaolong-Lian/ data-uni=uni0916>Xiaolong Lian</a>,
<a href=/gwf-publications/people/J/Jingjing-An/ data-uni=uni0917>Jingjing An</a>,
<a href=/gwf-publications/people/N/Ningbo-Geng/ data-uni=uni0447>Ningbo Geng</a>,
<a href=/gwf-publications/people/H/Haijun-Zhang/ data-uni=uni0918>Haijun Zhang</a>,
<a href=/gwf-publications/people/J/Jonathan-K-Challis/ data-uni=uni0019>Jonathan K. Challis</a>,
<a href=/gwf-publications/people/Y/Yun-Luo/ data-uni=uni0919>Yun Luo</a>,
<a href=/gwf-publications/people/Y/Ya-Xin-Liu/ data-uni=uni0920>Ya-Xin Liu</a>,
<a href=/gwf-publications/people/G/Guanyong-Su/ data-uni=uni0544>Guanyong Su</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/Y/Yingwen-Li/ data-uni=uni0334>Yingwen Li</a>,
<a href=/gwf-publications/people/Z/Zhihao-Liu/ data-uni=uni0921>Zhihao Liu</a>,
<a href=/gwf-publications/people/Y/Yanjun-Shen/ data-uni=uni0907>Yanjun Shen</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a>,
<a href=/gwf-publications/people/Y/Yufeng-Gong/ data-uni=uni0008>Yufeng Gong</a><br><a href=/gwf-publications/volumes/G21-40/ class=text-muted>Environmental Science & Technology</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-40001><div class="card-body p-3 small">Due to commercial uses and environmental degradation of aryl phosphate esters, diphenyl phosphate (DPhP) is frequently detected in environmental matrices and is thus of growing concern worldwide. However, information on potential adverse effects of chronic exposure to DPhP at environmentally realistic concentrations was lacking. Here, we investigated the effects of life cycle exposure to DPhP on zebrafish at environmentally relevant concentrations of 0.8, 3.9, or 35.6 μg/L and employed a dual-omics approach (metabolomics and transcriptomics) to characterize potential modes of action. Exposure to DPhP at 35.6 μg/L for 120 days resulted in significant reductions in body mass and length of male zebrafish, but did not cause those same effects to females. Predominant toxicological mechanisms, including inhibition of oxidative phosphorylation, down-regulation of fatty acid oxidation, and up-regulation of phosphatidylcholine degradation, were revealed by integrated dual-omics analysis and successfully linked to adverse outcomes. Activity of succinate dehydrogenase and protein content of carnitine O-palmitoyltransferase 1 were significantly decreased in livers of male fish exposed to DPhP, which further confirmed the proposed toxicological mechanisms. This study is the first to demonstrate that chronic, low-level exposure to DPhP can retard growth via inhibiting energy output in male zebrafish.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-41001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-41001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-41001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-41001/>Comparison of approaches to quantify SARS-CoV-2 in wastewater using RT-qPCR: Results and implications from a collaborative inter-laboratory study in Canada</a></strong><br><a href=/gwf-publications/people/A/Alex-HS-Chik/ data-uni=uni0922>Alex H.S. Chik</a>,
<a href=/gwf-publications/people/M/Melissa-B-Glier/ data-uni=uni0924>Melissa B. Glier</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=uni0466>Chand S. Mangat</a>,
<a href=/gwf-publications/people/X/Xiao-Li-Pang/ data-uni=uni0009>Xiao-Li Pang</a>,
<a href=/gwf-publications/people/Y/Yuanyuan-Qiu/ data-uni=uni0009>Yuanyuan Qiu</a>,
<a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=uni0026>Patrick M. D'Aoust</a>,
<a href=/gwf-publications/people/J/Jean-Baptiste-Burnet/ data-uni=uni0925>Jean-Baptiste Burnet</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0026>Robert Delatolla</a>,
<a href=/gwf-publications/people/S/Sarah-Dorner/ data-uni=uni0199>Sarah Dorner</a>,
<a href=/gwf-publications/people/Q/Qiudi-Geng/ data-uni=uni0146>Qiudi Geng</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a>,
<a href=/gwf-publications/people/R/Robert-M-McKay/ data-uni=uni0926>Robert M. McKay</a>,
<a href=/gwf-publications/people/M/Michael-R-Mulvey/ data-uni=uni0466>Michael R. Mulvey</a>,
<a href=/gwf-publications/people/N/Natalie-Prystajecky/ data-uni=uni0927>Natalie Prystajecky</a>,
<a href=/gwf-publications/people/N/Nivetha-Srikanthan/ data-uni=uni0002>Nivetha Srikanthan</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/B/Bernadette-Conant/ data-uni=uni0508>Bernadette Conant</a>,
<a href=/gwf-publications/people/S/Steve-E-Hrudey/ data-uni=uni0009>Steve E. Hrudey</a><br><a href=/gwf-publications/volumes/G21-41/ class=text-muted>Journal of Environmental Sciences, Volume 107</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-41001><div class="card-body p-3 small">Detection of SARS-CoV-2 RNA in wastewater is a promising tool for informing public health decisions during the COVID-19 pandemic. However, approaches for its analysis by use of reverse transcription quantitative polymerase chain reaction (RT-qPCR) are still far from standardized globally. To characterize inter- and intra-laboratory variability among results when using various methods deployed across Canada, aliquots from a real wastewater sample were spiked with surrogates of SARS-CoV-2 (gamma-radiation inactivated SARS-CoV-2 and human coronavirus strain 229E [HCoV-229E]) at low and high levels then provided "blind" to eight laboratories. Concentration estimates reported by individual laboratories were consistently within a 1.0-log10 range for aliquots of the same spiked condition. All laboratories distinguished between low- and high-spikes for both surrogates. As expected, greater variability was observed in the results amongst laboratories than within individual laboratories, but SARS-CoV-2 RNA concentration estimates for each spiked condition remained mostly within 1.0-log10 ranges. The no-spike wastewater aliquots provided yielded non-detects or trace levels (&lt;20 gene copies/mL) of SARS-CoV-2 RNA. Detections appear linked to methods that included or focused on the solids fraction of the wastewater matrix and might represent in-situ SARS-CoV-2 to the wastewater sample. HCoV-229E RNA was not detected in the no-spike aliquots. Overall, all methods yielded comparable results at the conditions tested. Partitioning behavior of SARS-CoV-2 and spiked surrogates in wastewater should be considered to evaluate method effectiveness. A consistent method and laboratory to explore wastewater SARS-CoV-2 temporal trends for a given system, with appropriate quality control protocols and documented in adequate detail should succeed.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-42001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-42001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-42001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-42001/>Responses of juvenile fathead minnow (Pimephales promelas) gut microbiome to a chronic dietary exposure of benzo[a]pyrene</a></strong><br><a href=/gwf-publications/people/A/Abigail-DeBofsky/ data-uni=uni0001>Abigail DeBofsky</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/J/Jonathan-K-Challis/ data-uni=uni0019>Jonathan K. Challis</a>,
<a href=/gwf-publications/people/N/Niteesh-Jain/ data-uni=uni0001>Niteesh Jain</a>,
<a href=/gwf-publications/people/M/Markus-Brinkmann/ data-uni=uni0001>Markus Brinkmann</a>,
<a href=/gwf-publications/people/P/Paul-W-Jones/ data-uni=uni0408>Paul W. Jones</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a><br><a href=/gwf-publications/volumes/G21-42/ class=text-muted>Environmental Pollution, Volume 278</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-42001><div class="card-body p-3 small">The microbiome has been described as an additional host “organ” with well-established beneficial roles. However, the effects of exposures to chemicals on both structure and function of the gut microbiome of fishes are understudied. To determine effects of benzo[ a ]pyrene (BaP), a model persistent organic pollutant, on structural shifts of gut microbiome in juvenile fathead minnows ( Pimephales promelas ), fish were exposed ad libitum in the diet to concentrations of 1, 10, 100, or 1000 μg BaP g −1 food, in addition to a vehicle control, for two weeks. To determine the link between exposure to BaP and changes in the microbial community, concentrations of metabolites of BaP were measured in fish bile and 16S rRNA amplicon sequencing was used to evaluate the microbiome. Exposure to BaP only reduced alpha-diversity at the greatest exposure concentrations. However, it did alter community composition assessed as differential abundance of taxa and reduced network complexity of the microbial community in all exposure groups. Results presented here illustrate that environmentally-relevant concentrations of BaP can alter the diversity of the gut microbiome and community network connectivity. Highlights • Dominant phyla of gut microbiome are consistent with those of other freshwater fishes. • BaP metabolites and exposure doses were consistent with those found in contaminated sites. • Dietary BaP exposure has significant, dose-dependent effects on the fish gut microbiome. • Dietary BaP exposure altered association networks of gut microbiome. Environmentally-relevant concentrations of BaP can alter the diversity of the gut microbiome and community network connectivity via dietary exposure route.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-44001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-44001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-44001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-44001/>Remodeling of Arctic char ( &lt;i>Salvelinus alpinus&lt;/i> ) lipidome under a stimulated scenario of Arctic warming</a></strong><br><a href=/gwf-publications/people/C/Chao-Wang/ data-uni=uni0037>Chao Wang</a>,
<a href=/gwf-publications/people/Y/Yufeng-Gong/ data-uni=uni0008>Yufeng Gong</a>,
<a href=/gwf-publications/people/F/Fuchang-Deng/ data-uni=uni0385>Fuchang Deng</a>,
<a href=/gwf-publications/people/E/Enmin-Ding/ data-uni=uni0928>Enmin Ding</a>,
<a href=/gwf-publications/people/J/Jie-Tang/ data-uni=uni0053>Jie Tang</a>,
<a href=/gwf-publications/people/G/Garry-Codling/ data-uni=uni1361>Garry Codling</a>,
<a href=/gwf-publications/people/J/Jonathan-K-Challis/ data-uni=uni0019>Jonathan K. Challis</a>,
<a href=/gwf-publications/people/D/Derek-Green/ data-uni=uni1344>Derek Green</a>,
<a href=/gwf-publications/people/J/Jing-Wang/ data-uni=uni0053>Jing Wang</a>,
<a href=/gwf-publications/people/Q/Qi-Liang-Chen/ data-uni=uni0915>Qi-Liang Chen</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/S/Shu-Su/ data-uni=uni0231>Shu Su</a>,
<a href=/gwf-publications/people/Z/Zilin-Yang/ data-uni=uni0929>Zilin Yang</a>,
<a href=/gwf-publications/people/J/Jason-C-Raine/ data-uni=uni0001>Jason C. Raine</a>,
<a href=/gwf-publications/people/P/Paul-W-Jones/ data-uni=uni0408>Paul W. Jones</a>,
<a href=/gwf-publications/people/S/Song-Tang/ data-uni=uni0385>Song Tang</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a><br><a href=/gwf-publications/volumes/G21-44/ class=text-muted>Global Change Biology, Volume 27, Issue 14</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-44001><div class="card-body p-3 small">Arctic warming associated with global climate change poses a significant threat to populations of wildlife in the Arctic. Since lipids play a vital role in adaptation of organisms to variations in temperature, high-resolution mass-spectrometry-based lipidomics can provide insights into adaptive responses of organisms to a warmer environment in the Arctic and help to illustrate potential novel roles of lipids in the process of thermal adaption. In this study, we studied an ecologically and economically important species-Arctic char (Salvelinus alpinus)-with a detailed multi-tissue analysis of the lipidome in response to chronic shifts in temperature using a validated lipidomics workflow. In addition, dynamic alterations in the hepatic lipidome during the time course of shifts in temperature were also characterized. Our results showed that early life stages of Arctic char were more susceptible to variations in temperature. One-year-old Arctic char responded to chronic increases in temperature with coordinated regulation of lipids, including headgroup-specific remodeling of acyl chains in glycerophospholipids (GP) and extensive alterations in composition of lipids in membranes, such as less lyso-GPs, and more ether-GPs and sphingomyelin. Glycerolipids (e.g., triacylglycerol, TG) also participated in adaptive responses of the lipidome of Arctic char. Eight-week-old Arctic char exhibited rapid adaptive alterations of the hepatic lipidome to stepwise decreases in temperature while showing blunted responses to gradual increases in temperature, implying an inability to adapt rapidly to warmer environments. Three common phosphatidylethanolamines (PEs) (PE 36:6|PE 16:1_20:5, PE 38:7|PE 16:1_22:6, and PE 40:7|PE 18:1_22:6) were finally identified as candidate lipid biomarkers for temperature shifts via machine learning approach. Overall, this work provides additional information to a better understanding of underlying regulatory mechanisms of the lipidome of Arctic organisms in the face of near-future warming.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-45001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-45001/>Environmental DNA of preservative ethanol performed better than water samples in detecting macroinvertebrate diversity using metabarcoding</a></strong><br><a href=/gwf-publications/people/Y/Yu-Wang/ data-uni=uni0053>Yu Wang</a>,
<a href=/gwf-publications/people/K/Kai-Chen/ data-uni=uni0380>Kai Chen</a>,
<a href=/gwf-publications/people/J/Jin-Gao/ data-uni=uni0447>Jin Gao</a>,
<a href=/gwf-publications/people/M/Meng-Wang/ data-uni=uni0384>Meng Wang</a>,
<a href=/gwf-publications/people/J/Jie-Dong/ data-uni=uni0930>Jie Dong</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a>,
<a href=/gwf-publications/people/X/Xiaowei-Jin/ data-uni=uni0931>Xiaowei Jin</a>,
<a href=/gwf-publications/people/B/Beixin-Wang/ data-uni=uni0932>Beixin Wang</a><br><a href=/gwf-publications/volumes/G21-45/ class=text-muted>Diversity and Distributions, Volume 27, Issue 10</a></span></p></div><h4>2020</h4><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G20-69001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G20-69001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G20-69001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G20-69001/>Differential responses of gut microbiota of male and female fathead minnow (Pimephales promelas) to a short-term environmentally-relevant, aqueous exposure to benzo[a]pyrene</a></strong><br><a href=/gwf-publications/people/A/Abigail-DeBofsky/ data-uni=uni0001>Abigail DeBofsky</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/C/Chelsea-Grimard/ data-uni=uni0001>Chelsea Grimard</a>,
<a href=/gwf-publications/people/A/Alper-James-G-Alcaraz/ data-uni=uni0001>Alper James G. Alcaraz</a>,
<a href=/gwf-publications/people/M/Markus-Brinkmann/ data-uni=uni0001>Markus Brinkmann</a>,
<a href=/gwf-publications/people/M/Markus-Hecker/ data-uni=uni0001>Markus Hecker</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a><br><a href=/gwf-publications/volumes/G20-69/ class=text-muted>Chemosphere, Volume 252</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G20-69001><div class="card-body p-3 small">In addition to aiding in digestion of food and uptake of nutrients, microbiota in guts of vertebrates are responsible for regulating several beneficial functions, including development of an organism and maintaining homeostasis. However, little is known about effects of exposures to chemicals on structure and function of gut microbiota of fishes. To assess effects of exposure to polycyclic aromatic hydrocarbons (PAHs) on gut microbiota, male and female fathead minnows ( Pimephales promelas ) were exposed to environmentally-relevant concentrations of the legacy PAH benzo[ a ]pyrene (BaP) in water. Measured concentrations of BaP ranged from 2.3 × 10 −3 to 1.3 μg L −1 . The community of microbiota in the gut were assessed by use of 16S rRNA metagenetics. Exposure to environmentally-relevant aqueous concentrations of BaP did not alter expression levels of mRNA for cyp1a1 , a “classic” biomarker of exposure to BaP, but resulted in shifts in relative compositions of gut microbiota in females rather than males. Results presented here illustrate that in addition to effects on more well-studied molecular endpoints, relative compositions of the microbiota in guts of fish can also quickly respond to exposure to chemicals, which can provide additional mechanisms for adverse effects on individuals. • Female and male fathead minnows exhibited significantly different gut microbiota. • Exposure to BaP altered structures in female gut microbiota, but not in males. • Exposure to BaP altered predicted functions in gut microbiota of fathead minnow. • Gut microbiome was more sensitive to a low dose BaP than host’s ahr1 and cyp1a1.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G20-95001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G20-95001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G20-95001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G20-95001/>Integrated assessment of west coast of South Korea by use of benthic bacterial community structure as determined by eDNA, concentrations of contaminants, and in vitro bioassays</a></strong><br><a href=/gwf-publications/people/A/Aslan-Hwanhwi-Lee/ data-uni=uni0001>Aslan Hwanhwi Lee</a>,
<a href=/gwf-publications/people/J/Jung-Hyun-Lee/ data-uni=uni0173>Jung-Hyun Lee</a>,
<a href=/gwf-publications/people/S/Seongjin-Hong/ data-uni=uni0300>Seongjin Hong</a>,
<a href=/gwf-publications/people/B/Bong-Oh-Kwon/ data-uni=uni1332>Bong-Oh Kwon</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a>,
<a href=/gwf-publications/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications/people/J/Jong-Seong-Khim/ data-uni=uni1021>Jong Seong Khim</a><br><a href=/gwf-publications/volumes/G20-95/ class=text-muted>Environment International, Volume 137</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G20-95001><div class="card-body p-3 small">During the past few decades, contamination of sediments by persistent toxic substances (PTSs) has been observed in estuarine and coastal areas on the west coast of South Korea. The contaminants are suspected to cause toxicities in aquatic biota, but little is known about their ecological effects, particularly on benthic microbial communities. In this study, an eDNA-based assessment was applied along with classic assessments of exposure, such as chemistry and in vitro bioassays, to evaluate condition of benthic bacterial communities subjected to PTSs. Two strategies were adopted for the study. One was to conduct a comprehensive assessment in space (by comparing seawater and freshwater sites at five coastal regions) and in time (by following change over a 5-y period). Although we found that bacterial composition varied among and within years, some phyla, such as Proteobacteria (28.7%), Actinobacteria (13.1%), Firmicutes (12.7%), and Chloroflexi (12.5%) were consistently dominated across the study regions. Certain bacterial groups, such as Firmicutes and Verrucomicrobia have been linked to contamination at some sites in the study area and at specific points in time. Bacterial communities were not significantly correlated with salinity or AhR- and ER-mediated potencies, whereas concentrations of PAHs, APs, and certain metals (Cd and Hg) exhibited significant associations to the structure of bacterial communities at the phylum level. In fact, the relative abundance of microbes in the phylum Planctomycetes was significantly and negatively correlated with concentrations of PAHs and metals. Thus, the relative abundance of Planctomycetes could be used as an indicator of sedimentary contamination by PAHs and/or metals. Based on our correlation analyses, Cd and ER-mediated potencies were associated more with bacterial abundances at the class taxonomic level than were other PTSs and metals. Overall, the eDNA-based assessment was useful by augmenting more traditional measures of exposure and responses in a sediment triad approach and has potential as a more rapid screening tool.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G20-172001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G20-172001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G20-172001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G20-172001/>Effects of the husky oil spill on gut microbiota of native fishes in the North Saskatchewan River, Canada</a></strong><br><a href=/gwf-publications/people/A/Abigail-DeBofsky/ data-uni=uni0001>Abigail DeBofsky</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/T/Timothy-D-Jardine/ data-uni=uni0001>Timothy D. Jardine</a>,
<a href=/gwf-publications/people/J/Janet-E-Hill/ data-uni=uni0001>Janet E. Hill</a>,
<a href=/gwf-publications/people/P/Paul-W-Jones/ data-uni=uni0408>Paul W. Jones</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a><br><a href=/gwf-publications/volumes/G20-172/ class=text-muted>Aquatic Toxicology, Volume 229</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G20-172001><div class="card-body p-3 small">• Concentrations of PAHs in muscle suggests continued exposure to the residual spilled oil. • Identity of the host species was the dominant driver in shaping the gut microbiome of fish. • Structures of gut microbiomes were correlated with concentrations of PAHs in muscle in walleye. In July 2016, a Husky Energy pipeline spilled 225,000 L of diluted heavy crude oil, with a portion of the oil entering the North Saskatchewan River near Maidstone, SK, Canada. This event provided a unique opportunity to assess potential effects of a crude oil constituent (namely polycyclic aromatic hydrocarbons, PAHs) on a possible sensitive indicator of freshwater ecosystem health, the gut microbiota of native fishes. In summer 2017, goldeye ( Hiodon alosoides ), walleye ( Sander vitreus ), northern pike ( Esox lucius ), and shorthead redhorse ( Moxostoma macrolepidotum ) were collected at six locations upstream and downstream of the spill. Muscle and bile were collected from individual fish for quantification of PAHs and intestinal contents were collected for characterization of the microbial community of the gut. Results suggested that host species is a significant determinant of gut microbiota, with significant differences among the species across sites. Concentrations of PAHs in dorsal muscle were significantly correlated with gut community compositions of walleye, but not of the other fishes. Concentrations of PAHs in muscle were also correlated with abundances of several families of bacteria among fishes. This study represents one of the first to investigate the response of the gut microbiome of wild fishes to chemical stressors.</div></div><h4>2019</h4><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G19-92001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G19-92001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G19-92001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G19-92001/>Sedimentary DNA reveals over 150 years of ecosystem change by human activities in Lake Chao, China</a></strong><br><a href=/gwf-publications/people/F/Fei-Long-Li/ data-uni=uni1212>Fei-Long Li</a>,
<a href=/gwf-publications/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/J/Ji-Zhong-Wang/ data-uni=uni1482>Ji-Zhong Wang</a><br><a href=/gwf-publications/volumes/G19-92/ class=text-muted>Environment International, Volume 133</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G19-92001><div class="card-body p-3 small">Understanding the extent and directionality of the impact of human activities on ecosystems is directly related to their management and protection. However, the lack of historical data limits our understanding of ecosystem changes with long-term exposure to human activities. Recently, lake sedimentary DNA (sedDNA) has become a powerful tool for revealing changes in ecosystems at the century and millennium scales. Here, we used sedDNA to reveal the dynamic of the microbial community (including bacteria and micro-eukaryotes) in Lake Chao over the past 150 years, and further explored the effects of long-term nutrient and heavy metal loads on these communities. Our data show that nutrient and heavy metal loads in Lake Chao have increased by ca. 2 to 4-fold since the 1960s. In response, the community structure, diversity, and ecological network of bacteria and micro-eukaryotes changed significantly during the 1960s, the 1980s and the 2010s. Importantly, community structure was more sensitive to human activities than diversity. We also found that the relative abundance of some taxa associated with nitrification and algal blooms (e.g., taxa in Nitrospira sp., Peridinales) has increased ca. 100-fold since the 1960s. Nutrient could better explain the variation in the bacterial community (ca. twice as much as heavy metal), while heavy metal explained micro-eukaryotes better (ca. 3 or 5-fold as much as nutrient). In particular, based on parsimonious models from distance-based linear model (distLM), we further identified that Pb is the key factor affecting the bacterial and micro-eukaryotes community in Lake Chao in addition to nutrient. Our study reveals the impacts of long-term human activities on lake ecosystems from multiple perspectives of nutrient and heavy metal loads, community structure, diversity and ecological network, these findings will contribute to the management and conservation of lakes in the future.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G19-113001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G19-113001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G19-113001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G19-113001/>Selenium oxyanion bioconcentration in natural freshwater periphyton</a></strong><br><a href=/gwf-publications/people/B/Blue-E-Markwart/ data-uni=uni0001>Blue E. Markwart</a>,
<a href=/gwf-publications/people/K/Karsten-Liber/ data-uni=uni0001>Karsten Liber</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/K/Katherine-Raes/ data-uni=uni0001>Katherine Raes</a>,
<a href=/gwf-publications/people/M/Markus-Hecker/ data-uni=uni0001>Markus Hecker</a>,
<a href=/gwf-publications/people/D/David-M-Janz/ data-uni=uni0001>David M. Janz</a>,
<a href=/gwf-publications/people/L/Lorne-E-Doig/ data-uni=uni0001>Lorne E. Doig</a><br><a href=/gwf-publications/volumes/G19-113/ class=text-muted>Ecotoxicology and Environmental Safety, Volume 180</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G19-113001><div class="card-body p-3 small">Selenium (Se) enrichment has been demonstrated to vary by several orders of magnitude among species of planktonic algae. This is a substantial source of uncertainty when modelling Se biodynamics in aquatic systems. In addition, Se bioconcentration data are largely lacking for periphytic species of algae, and for multi-species periphyton biofilms, adding to the challenge of modelling Se transfer in periphyton-based food webs. To better predict Se dynamics in periphyton dominated, freshwater ecosystems, the goal of this study was to assess the relative influence of periphyton community composition on the uptake of waterborne Se oxyanions. Naturally grown freshwater periphyton communities, sampled from five different water bodies, were exposed to environmentally relevant concentrations of selenite [Se(IV)] or selenate [Se(VI)] (nominal concentrations of 5 and 25 μg Se L-1) under similar, controlled laboratory conditions for a period of 8 days. Unique periphyton assemblages were derived from the five different field sites, as confirmed by light microscopy and targeted DNA sequencing of the plastid 23S rRNA gene in algae. Selenium accumulation demonstrated a maximum of 23.6-fold difference for Se(IV) enrichment and 2.1-fold difference for Se(VI) enrichment across the periphyton/biofilm assemblages tested. The assemblage from one field site demonstrated both high accumulation of Se(IV) and iron, and was subjected to additional experimentation to elucidate the mechanism(s) of Se accumulation. Selenite accumulation (at nominal concentrations of 5 and 25 μg Se L-1 and mean pH of 7.5 across all treatment replicates) was assessed in both unaltered and heat-killed periphyton, and in periphyton from the same site grown without light to exclude phototrophic organisms. Following an exposure length of 8 days, all periphyton treatments showed similar levels of Se accumulation, indicating that much of the apparent uptake of Se(IV) was due to non-biological processes (i.e., surface adsorption). The results of this study will help reduce uncertainty in the prediction of Se dynamics and food-chain transfer in freshwater environments. Further exploration of the ecological consequences of extracellular adsorption of Se(IV) to periphyton, rather than intracellular absorption, is recommended to further refine predictions related to Se biodynamics in freshwater food webs.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G19-188001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G19-188001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G19-188001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G19-188001/>Acid mine drainage affects the diversity and metal resistance gene profile of sediment bacterial community along a river</a></strong><br><a href=/gwf-publications/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications/people/S/Song-Tang/ data-uni=uni0385>Song Tang</a>,
<a href=/gwf-publications/people/M/Mao-Wang/ data-uni=uni1292>Mao Wang</a>,
<a href=/gwf-publications/people/W/Weimin-Sun/ data-uni=uni1293>Weimin Sun</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/H/Hui-Peng/ data-uni=uni0420>Hui Peng</a>,
<a href=/gwf-publications/people/A/Ai-Min-Zhong/ data-uni=uni0231>Ai-Min Zhong</a>,
<a href=/gwf-publications/people/H/Hongling-Liu/ data-uni=uni0153>Hongling Liu</a>,
<a href=/gwf-publications/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications/people/H/Hongxia-Yu/ data-uni=uni0153>Hongxia Yu</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a>,
<a href=/gwf-publications/people/M/Markus-Hecker/ data-uni=uni0001>Markus Hecker</a><br><a href=/gwf-publications/volumes/G19-188/ class=text-muted>Chemosphere, Volume 217</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G19-188001><div class="card-body p-3 small">Acid mine drainage (AMD) is one of the most hazardous byproducts of some types of mining. However, research on how AMD affects the bacterial community structure of downstream riverine ecosystems and the distribution of metal resistance genes (MRGs) along pollution gradient is limited. Comprehensive geochemical and high-throughput next-generation sequencing analyses can be integrated to characterize spatial distributions and MRG profiles of sediment bacteria communities along the AMD-contaminated Hengshi River. We found that (1) diversities of bacterial communities significantly and gradually increased along the river with decreasing contamination, suggesting community composition reflected changes in geochemical conditions; (2) relative abundances of phyla Proteobacteria and genus Halomonas and Planococcaceae that function in metal reduction decreased along the AMD gradient; (3) low levels of sediment salinity, sulfate, aquatic lead (Pb), and cadmium (Cd) were negatively correlated with bacterial diversity despite pH was in a positive manner with diversity; and (4) arsenic (As) and copper (Cu) resistance genes corresponded to sediment concentrations of As and Cu, respectively. Altogether, our findings offer initial insight into the distribution patterns of sediment bacterial community structure, diversity and MRGs along a lotic ecosystem contaminated by AMD, and the factors that affect them.</div></div><h4>2018</h4><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G18-67001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G18-67001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G18-67001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G18-67001/>Application of Environmental DNA Metabarcoding for Predicting Anthropogenic Pollution in Rivers</a></strong><br><a href=/gwf-publications/people/F/Fei-Long-Li/ data-uni=uni1212>Fei-Long Li</a>,
<a href=/gwf-publications/people/Y/Ying-Peng/ data-uni=uni1213>Ying Peng</a>,
<a href=/gwf-publications/people/W/Wendi-Fang/ data-uni=uni0153>Wendi Fang</a>,
<a href=/gwf-publications/people/F/Florian-Altermatt/ data-uni=uni0099>Florian Altermatt</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/J/Jianghua-Yang/ data-uni=uni0153>Jianghua Yang</a>,
<a href=/gwf-publications/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a><br><a href=/gwf-publications/volumes/G18-67/ class=text-muted>Environmental Science & Technology</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G18-67001><div class="card-body p-3 small">Rivers are among the most threatened freshwater ecosystems, and anthropogenic activities are affecting both river structures and water quality. While assessing the organisms can provide a comprehensive measure of a river's ecological status, it is limited by the traditional morphotaxonomy-based biomonitoring. Recent advances in environmental DNA (eDNA) metabarcoding allow to identify prokaryotes and eukaryotes in one sequencing run, and could thus allow unprecedented resolution. Whether such eDNA-based data can be used directly to predict the pollution status of rivers as a complementation of environmental data remains unknown. Here we used eDNA metabarcoding to explore the main stressors of rivers along which community structure changes, and to identify the method's potential for predicting pollution status based on eDNA data. We showed that a broad range of taxa in bacterial, protistan, and metazoan communities could be profiled with eDNA. Nutrients were the main driving stressor affecting communities' structure, alpha diversity, and the ecological network. We specifically observed that the relative abundance of indicative OTUs was significantly correlated with nutrient levels. These OTUs data could be used to predict the nutrient status up to 79% accuracy on testing data sets. Thus, our study gives a novel approach to predicting the pollution status of rivers by eDNA data.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G18-131001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G18-131001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G18-131001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G18-131001/>In situ microbiota distinguished primary anthropogenic stressor in freshwater sediments</a></strong><br><a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/T/Tilman-Floehr/ data-uni=uni0113>Tilman Floehr</a>,
<a href=/gwf-publications/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications/people/H/Hongxia-Xiao/ data-uni=uni0113>Hongxia Xiao</a>,
<a href=/gwf-publications/people/J/Jianghua-Yang/ data-uni=uni0153>Jianghua Yang</a>,
<a href=/gwf-publications/people/P/Pu-Xia/ data-uni=uni1018>Pu Xia</a>,
<a href=/gwf-publications/people/G/G-Allen-Burton/ data-uni=uni0017>G. Allen Burton</a>,
<a href=/gwf-publications/people/H/Henner-Hollert/ data-uni=uni0309>Henner Hollert</a><br><a href=/gwf-publications/volumes/G18-131/ class=text-muted>Environmental Pollution, Volume 239</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G18-131001><div class="card-body p-3 small">Conventional assessment and evaluation of sediment quality are based on laboratory-based ecotoxicological and chemical measurements with lack of concern for ecological relevance. Microbiotas in sediment are responsive to pollutants and can be used as alternative ecological indicators of sediment pollutants; however, the linkage between the microbial ecology and ecotoxicological endpoints in response to sediment contamination has been poorly evaluated. Here, in situ microbiotas from the Three Gorges Reservoir (TGR) area of the Yangtze River were characterized by DNA metabarcoding approaches, and then, changes of in situ microbiotas were compared with the ecotoxicological endpoint, aryl hydrocarbon receptor (AhR) mediated activity, and level of polycyclic aromatic hydrocarbons (PAHs) in sediments. PAHs and organic pollutant mixtures mediating AhR activity had different effects on the structures of microbiotas. Specifically, Shannon indices of protistan communities were negatively correlated with the levels of AhR mediated activity and PAHs. The sediment AhR activity was positively correlated with the relative abundance of prokaryotic Acetobacteraceae, but had a negative correlation with protistan Oxytrichidae. Furthermore, a quantitative classification model was built to predict the level of AhR activity based on the relative abundances of Acetobacteraceae and Oxytrichidae. These results suggested that in situ Protista communities could provide a useful tool for monitoring and assessing ecological stressors. The observed responses of microbial community provided supplementary evidence to support that the AhR-active pollutants, such as PAHs, were the primary stressors of the aquatic community in TGR area.</div></div><div data-prj=prj6><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G18-132001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G18-132001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G18-132001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G18-132001/>eDNA-based bioassessment of coastal sediments impacted by an oil spill</a></strong><br><a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0001>Yuwei Xie</a>,
<a href=/gwf-publications/people/J/Jun-Xu/ data-uni=uni1214>Jun Xu</a>,
<a href=/gwf-publications/people/J/Jianghua-Yang/ data-uni=uni0153>Jianghua Yang</a>,
<a href=/gwf-publications/people/S/Seon-Jin-Kim/ data-uni=uni0591>Seon-Jin Kim</a>,
<a href=/gwf-publications/people/S/Seongjin-Hong/ data-uni=uni0300>Seongjin Hong</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0106>John P. Giesy</a>,
<a href=/gwf-publications/people/U/Un-Hyuk-Yim/ data-uni=uni1019>Un Hyuk Yim</a>,
<a href=/gwf-publications/people/W/Won-Heum-Shim/ data-uni=uni1020>Won Heum Shim</a>,
<a href=/gwf-publications/people/H/Hongxia-Yu/ data-uni=uni0153>Hongxia Yu</a>,
<a href=/gwf-publications/people/J/Jong-Seong-Khim/ data-uni=uni1021>Jong Seong Khim</a><br><a href=/gwf-publications/volumes/G18-132/ class=text-muted>Environmental Pollution, Volume 238</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G18-132001><div class="card-body p-3 small">Oil spills offshore can cause long-term ecological effects on coastal marine ecosystems. Despite their important ecological roles in the cycling of energy and nutrients in food webs, effects on bacteria, protists or arthropods are often neglected. Environmental DNA (eDNA) metabarcoding was applied to characterize changes in the structure of micro- and macro-biota communities of surface sediments over a 7-year period since the occurrence of Hebei Spirit oil spill on December 7, 2007. Alterations in diversities and structures of micro- and macro-biota were observed in the contaminated area where concentrations of polycyclic aromatic hydrocarbons were greater. Successions of bacterial, protists and metazoan communities revealed long-term ecological effects of residual oil. Residual oil dominated the largest cluster of the community-environment association network. Presence of bacterial families (Aerococcaceae and Carnobacteriaceae) and the protozoan family (Platyophryidae) might have conferred sensitivity of communities to oil pollution. Hydrocarbon-degrading bacterial families (Anaerolinaceae, Desulfobacteraceae, Helicobacteraceae and Piscirickettsiaceae) and algal family (Araphid pennate) were resistant to adverse effects of spilt oil. The protistan family (Subulatomonas) and arthropod families (Folsomia, Sarcophagidae Opomyzoidea, and Anomura) appeared to be positively associated with residual oil pollution. eDNA metabarcoding can provide a powerful tool for assessing effects of anthropogenic pollution, such as oil spills on sediment communities and its long-term trends in coastal marine environments.</div></div></div><div class=col-lg-3><a class="btn btn-lg btn-secondary btn-block mb-2" href="https://www.semanticscholar.org/search?q=Yuwei+Xie" title="Search for 'Yuwei Xie' on Semantic Scholar"><i class="ai ai-semantic-scholar"></i><span class=pl-sm-2>Search</span></a><div class=row><div class="col-12 col-md-6 col-lg-12"><div class=card><h5 class=card-header>Co-authors</h5><ul class="list-group list-group-flush list-group-compact"><li class=list-group-item><a href=/gwf-publications/people/J/John-P-Giesy/ class=align-middle data-uni=uni0106>John P. Giesy</a>
<span class="badge badge-secondary align-middle ml-2">16</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jun-Xu/ class=align-middle data-uni=uni1214>Jun Xu</a>
<span class="badge badge-secondary align-middle ml-2">7</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Paul-W-Jones/ class=align-middle data-uni=uni0408>Paul W. Jones</a>
<span class="badge badge-secondary align-middle ml-2">7</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Abigail-DeBofsky/ class=align-middle data-uni=uni0001>Abigail DeBofsky</a>
<span class="badge badge-secondary align-middle ml-2">6</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jonathan-K-Challis/ class=align-middle data-uni=uni0019>Jonathan K. Challis</a>
<span class="badge badge-secondary align-middle ml-2">6</span></li><li class="list-group-item list-group-toggle-btn py-1" data-toggle=collapse data-target=#more-coauthors aria-expanded=false aria-controls=more-coauthors>show all...</li><div class="collapse border-top" id=more-coauthors><li class=list-group-item><a href=/gwf-publications/people/M/Markus-Hecker/ class=align-middle data-uni=uni0001>Markus Hecker</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Markus-Brinkmann/ class=align-middle data-uni=uni0001>Markus Brinkmann</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Phillip-J-Ankley/ class=align-middle data-uni=uni0001>Phillip J. Ankley</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jianghua-Yang/ class=align-middle data-uni=uni0153>Jianghua Yang</a>
<span class="badge badge-secondary align-middle ml-2">3</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yufeng-Gong/ class=align-middle data-uni=uni0008>Yufeng Gong</a>
<span class="badge badge-secondary align-middle ml-2">3</span></li><li class=list-group-item><a href=/gwf-publications/people/F/Fei-Long-Li/ class=align-middle data-uni=uni1212>Fei-Long Li</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Seongjin-Hong/ class=align-middle data-uni=uni0300>Seongjin Hong</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Hongxia-Yu/ class=align-middle data-uni=uni0153>Hongxia Yu</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jong-Seong-Khim/ class=align-middle data-uni=uni1021>Jong Seong Khim</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Song-Tang/ class=align-middle data-uni=uni0385>Song Tang</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/V/Vince-P-Palace/ class=align-middle data-uni=uni0013>Vince P. Palace</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/Q/Qi-Liang-Chen/ class=align-middle data-uni=uni0915>Qi-Liang Chen</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mark-R-Servos/ class=align-middle data-uni=uni0002>Mark R. Servos</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Chand-S-Mangat/ class=align-middle data-uni=uni0466>Chand S. Mangat</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Chao-Wang/ class=align-middle data-uni=uni0037>Chao Wang</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jie-Tang/ class=align-middle data-uni=uni0053>Jie Tang</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Shu-Su/ class=align-middle data-uni=uni0231>Shu Su</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/Z/Zilin-Yang/ class=align-middle data-uni=uni0929>Zilin Yang</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jason-C-Raine/ class=align-middle data-uni=uni0001>Jason C. Raine</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Ying-Peng/ class=align-middle data-uni=uni1213>Ying Peng</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/W/Wendi-Fang/ class=align-middle data-uni=uni0153>Wendi Fang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/F/Florian-Altermatt/ class=align-middle data-uni=uni0099>Florian Altermatt</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/T/Tilman-Floehr/ class=align-middle data-uni=uni0113>Tilman Floehr</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Hongxia-Xiao/ class=align-middle data-uni=uni0113>Hongxia Xiao</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Pu-Xia/ class=align-middle data-uni=uni1018>Pu Xia</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/G/G-Allen-Burton/ class=align-middle data-uni=uni0017>G. Allen Burton</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Henner-Hollert/ class=align-middle data-uni=uni0309>Henner Hollert</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Seon-Jin-Kim/ class=align-middle data-uni=uni0591>Seon-Jin Kim</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/U/Un-Hyuk-Yim/ class=align-middle data-uni=uni1019>Un Hyuk Yim</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/W/Won-Heum-Shim/ class=align-middle data-uni=uni1020>Won Heum Shim</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Ji-Zhong-Wang/ class=align-middle data-uni=uni1482>Ji-Zhong Wang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/B/Blue-E-Markwart/ class=align-middle data-uni=uni0001>Blue E. Markwart</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Karsten-Liber/ class=align-middle data-uni=uni0001>Karsten Liber</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Katherine-Raes/ class=align-middle data-uni=uni0001>Katherine Raes</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/D/David-M-Janz/ class=align-middle data-uni=uni0001>David M. Janz</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/L/Lorne-E-Doig/ class=align-middle data-uni=uni0001>Lorne E. Doig</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mao-Wang/ class=align-middle data-uni=uni1292>Mao Wang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/W/Weimin-Sun/ class=align-middle data-uni=uni1293>Weimin Sun</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Hui-Peng/ class=align-middle data-uni=uni0420>Hui Peng</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Ai-Min-Zhong/ class=align-middle data-uni=uni0231>Ai-Min Zhong</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Hongling-Liu/ class=align-middle data-uni=uni0153>Hongling Liu</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Chelsea-Grimard/ class=align-middle data-uni=uni0001>Chelsea Grimard</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Alper-James-G-Alcaraz/ class=align-middle data-uni=uni0001>Alper James G. Alcaraz</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Aslan-Hwanhwi-Lee/ class=align-middle data-uni=uni0001>Aslan Hwanhwi Lee</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jung-Hyun-Lee/ class=align-middle data-uni=uni0173>Jung-Hyun Lee</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/B/Bong-Oh-Kwon/ class=align-middle data-uni=uni1332>Bong-Oh Kwon</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/T/Timothy-D-Jardine/ class=align-middle data-uni=uni0001>Timothy D. Jardine</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Janet-E-Hill/ class=align-middle data-uni=uni0001>Janet E. Hill</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/T/Tyler-Black/ class=align-middle data-uni=uni0029>Tyler Black</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/McKenzie-Perry/ class=align-middle data-uni=uni0013>McKenzie Perry</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mike-Paterson/ class=align-middle data-uni=uni0913>Mike Paterson</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mark-A-Hanson/ class=align-middle data-uni=uni0914>Mark A. Hanson</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Scott-N-Higgins/ class=align-middle data-uni=uni0283>Scott N. Higgins</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/X/Xiaolong-Lian/ class=align-middle data-uni=uni0916>Xiaolong Lian</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jingjing-An/ class=align-middle data-uni=uni0917>Jingjing An</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Ningbo-Geng/ class=align-middle data-uni=uni0447>Ningbo Geng</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Haijun-Zhang/ class=align-middle data-uni=uni0918>Haijun Zhang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yun-Luo/ class=align-middle data-uni=uni0919>Yun Luo</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Ya-Xin-Liu/ class=align-middle data-uni=uni0920>Ya-Xin Liu</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/G/Guanyong-Su/ class=align-middle data-uni=uni0544>Guanyong Su</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yingwen-Li/ class=align-middle data-uni=uni0334>Yingwen Li</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Z/Zhihao-Liu/ class=align-middle data-uni=uni0921>Zhihao Liu</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yanjun-Shen/ class=align-middle data-uni=uni0907>Yanjun Shen</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Alex-HS-Chik/ class=align-middle data-uni=uni0922>Alex H.S. Chik</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Melissa-B-Glier/ class=align-middle data-uni=uni0924>Melissa B. Glier</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/X/Xiao-Li-Pang/ class=align-middle data-uni=uni0009>Xiao-Li Pang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yuanyuan-Qiu/ class=align-middle data-uni=uni0009>Yuanyuan Qiu</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Patrick-M-DAoust/ class=align-middle data-uni=uni0026>Patrick M. D'Aoust</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jean-Baptiste-Burnet/ class=align-middle data-uni=uni0925>Jean-Baptiste Burnet</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/R/Robert-Delatolla/ class=align-middle data-uni=uni0026>Robert Delatolla</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Sarah-Dorner/ class=align-middle data-uni=uni0199>Sarah Dorner</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Q/Qiudi-Geng/ class=align-middle data-uni=uni0146>Qiudi Geng</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/R/Robert-M-McKay/ class=align-middle data-uni=uni0926>Robert M. McKay</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Michael-R-Mulvey/ class=align-middle data-uni=uni0466>Michael R. Mulvey</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Natalie-Prystajecky/ class=align-middle data-uni=uni0927>Natalie Prystajecky</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Nivetha-Srikanthan/ class=align-middle data-uni=uni0002>Nivetha Srikanthan</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/B/Bernadette-Conant/ class=align-middle data-uni=uni0508>Bernadette Conant</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Steve-E-Hrudey/ class=align-middle data-uni=uni0009>Steve E. Hrudey</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Niteesh-Jain/ class=align-middle data-uni=uni0001>Niteesh Jain</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/F/Fuchang-Deng/ class=align-middle data-uni=uni0385>Fuchang Deng</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/E/Enmin-Ding/ class=align-middle data-uni=uni0928>Enmin Ding</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/G/Garry-Codling/ class=align-middle data-uni=uni1361>Garry Codling</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/D/Derek-Green/ class=align-middle data-uni=uni1344>Derek Green</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jing-Wang/ class=align-middle data-uni=uni0053>Jing Wang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yu-Wang/ class=align-middle data-uni=uni0053>Yu Wang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Kai-Chen/ class=align-middle data-uni=uni0380>Kai Chen</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jin-Gao/ class=align-middle data-uni=uni0447>Jin Gao</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Meng-Wang/ class=align-middle data-uni=uni0384>Meng Wang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jie-Dong/ class=align-middle data-uni=uni0930>Jie Dong</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/X/Xiaowei-Jin/ class=align-middle data-uni=uni0931>Xiaowei Jin</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/B/Beixin-Wang/ class=align-middle data-uni=uni0932>Beixin Wang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Sonya-M-Havens/ class=align-middle data-uni=uni0082>Sonya M. Havens</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/L/Lisa-E-Peters/ class=align-middle data-uni=uni0013>Lisa E. Peters</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/L/Lauren-Timlick/ class=align-middle data-uni=uni0013>Lauren Timlick</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jos%C3%A9-Luis-Rodr%C3%ADguez-Gil/ class=align-middle data-uni=uni0283>José Luis Rodríguez-Gil</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/X/Xiaoli-Zhao/ class=align-middle data-uni=uni0864>Xiaoli Zhao</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Alana-Suevelin-Weber/ class=align-middle data-uni=uni0865>Alana Suevelin Weber</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/F/Femi-Francis-Oloye/ class=align-middle data-uni=uni0866>Femi Francis Oloye</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mohsen-Asadi/ class=align-middle data-uni=uni0439>Mohsen Asadi</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jenna-Cantin/ class=align-middle data-uni=uni0019>Jenna Cantin</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Kerry-N-McPhedran/ class=align-middle data-uni=uni0006>Kerry N. McPhedran</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Natacha-S-Hogan/ class=align-middle data-uni=uni0001>Natacha S. Hogan</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mike-Sadowski/ class=align-middle data-uni=uni0867>Mike Sadowski</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Chrystal-Landgraff/ class=align-middle data-uni=uni0005>Chrystal Landgraff</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li></div></ul></div></div><div class="col-12 col-md-6 col-lg-12"><div class="card my-2 my-md-0 my-lg-2"><h5 class=card-header>Venues</h5><ul class="list-group list-group-flush list-group-compact"><li class=list-group-item><a href=/gwf-publications/venues/gwf/ class=align-middle>GWF</a><span class="badge badge-secondary align-middle ml-2">20</span></li></ul></div></div></div></div></div></section></div><footer class="bg-gradient-light py-2 py-xl-3 mt-3 mt-md-4 mt-xl-5"><div class=container><p class="text-muted small px-1">Global Water Futures Publications!</p></div></footer><script src=https://code.jquery.com/jquery-3.3.1.slim.min.js integrity=sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js integrity=sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut crossorigin=anonymous></script><script src=https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js integrity=sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k crossorigin=anonymous></script><script>$(function(){$('[data-toggle="tooltip"]').tooltip();if($("#toggle-all-abstracts")){$("#toggle-all-abstracts").click(function(){var target=$("#toggle-all-abstracts");target.attr("disabled",true);if(target.attr("data-toggle-state")=="hide"){$(".abstract-collapse").collapse('show');target.attr("data-toggle-state","show");}else{$(".abstract-collapse").collapse('hide');target.attr("data-toggle-state","hide");}
target.attr("disabled",false);});$("#toggle-all-abstracts").attr("disabled",false);}})</script></body></html>